<!DOCTYPE html><html><head><link href='https://fonts.googleapis.com/css?family=Spectral' rel='stylesheet'><style>body {    font-family: 'Spectral';font-size: 14px;}</style></head><body><b>Search Term: </b>("sickle cell disease" OR "beta thalassemia" OR "beta-thalassemia") AND ("gene therapy" OR "genome editing" OR "mRNA therapy" OR ("hematopoietic stem cells" AND CRISPR AND (AAV OR "adeno-associated virus"))<br><b>Date run: </b>2020/01/07<br><b>Results recency: </b>2019/01/07<br><br>Jump to: <a href="#pubmed">PubMed Articles</a> | <a href="#companies">Companies</a> | <a href="#patents">Patents</a><br><a name="pubmed"></a><h2>PubMed Articles</h2> <br><br><b>Title:</b> Perspectives of Sickle Cell Disease Stakeholders on Heritable Genome Editing.<br><b>Abstract:</b> Advances in CRISPR technology and the announcement of the first gene-edited babies have sparked a global dialogue about the future of heritable genome editing (HGE). There has been an international call for public input to inform a substantive debate about benefits and risks of HGE. This study investigates the views of the sickle cell disease (SCD) community. We utilized a mixed-methods approach to examine SCD stakeholders' views in the United States. We found SCD stakeholders hold a nuanced view of HGE. Assuming the technology is shown to be safe and effective, they are just as supportive of HGE as genetics professionals, but more supportive than the general public. However, they are also concerned about the potential implications of HGE, despite this support. As discourse surrounding HGE advances, it is crucial to engage disease communities and other key stakeholders whose lives could be altered by these interventions.<br><b>Publication date:</b> 2019-11-19<br><b>Authors:</b> Brittany M Hollister, Mariclare C Gatter, Khadijah E Abdallah, Alyssa J Armsby, Ashley J Buscetta, Yen Ji Julia Byeon, Kayla E Cooper, Stacy Desine, Anitra Persaud, Kelly E Ormond, Vence L Bonham<br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 1.0<br><b>Relevance score 2:</b> 0.27<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31742431">Link</a></b><br><br><b>Title:</b> Menstrual blood-derived stem cells: toward therapeutic mechanisms, novel strategies, and future perspectives in the treatment of diseases.<br><b>Abstract:</b> Menstrual blood-derived stem cells (MenSCs) have great potential in the treatment of various diseases. As a novel type of mesenchymal stem cells (MSCs), MenSCs have attracted more interest due to their therapeutic effects in both animal models and clinical trials. Here, we described the differentiation, immunomodulation, paracrine, homing, and engraftment mechanisms of MenSCs. These include differentiation into targeting cells, immunomodulation with various immune cells, the paracrine effect on secreting cytokines, and homing and engraftment into injured sites. To better conduct MenSC-based therapy, some novel hotspots were proposed such as CRISPR (clustered regularly interspaced short palindromic repeats)/cas9-mediated gene modification, exosomes for cell-free therapy, single-cell RNA sequence for precision medicine, engineered MenSC-based therapy for the delivery platform, and stem cell niches for improving microenvironment. Subsequently, current challenges were elaborated on, with regard to age of donor, dose of MenSCs, transplantation route, and monitoring time. The management of clinical research with respect to MenSC-based therapy in diseases will become more normative and strict. Thus, a more comprehensive horizon should be considered that includes a combination of traditional solutions and novel strategies. In summary, MenSC-based treatment has a great potential in treating diseases through diverse strategies, and more therapeutic mechanisms and novel strategies need to be elucidated for future regenerative medicine and clinical applications.<br><b>Publication date:</b> 2019-12-22<br><b>Authors:</b> Lijun Chen, Jingjing Qu, Tianli Cheng, Xin Chen, Charlie Xiang<br><b>Journal:</b> Stem Cell Res Ther<br><b>ISSN:</b> 1757-6512<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.24<br><b>Two-year IF:</b> 4.66<br><b>SJR:</b> 1.548<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31864423">Link</a></b><br><br><b>Title:</b> Patient and Disease-Specific Induced Pluripotent Stem Cells for Discovery of Personalized Cardiovascular Drugs and Therapeutics.<br><b>Abstract:</b> Human induced pluripotent stem cells (iPSCs) have emerged as an effective platform for regenerative therapy, disease modeling, and drug discovery. iPSCs allow for the production of limitless supply of patient-specific somatic cells that enable advancement in cardiovascular precision medicine. Over the past decade, researchers have developed protocols to differentiate iPSCs to multiple cardiovascular lineages, as well as to enhance the maturity and functionality of these cells. Despite significant advances, drug therapy and discovery for cardiovascular disease have lagged behind other fields such as oncology. We speculate that this paucity of drug discovery is due to a previous lack of efficient, reproducible, and translational model systems. Notably, existing drug discovery and testing platforms rely on animal studies and clinical trials, but investigations in animal models have inherent limitations due to interspecies differences. Moreover, clinical trials are inherently flawed by assuming that all individuals with a disease will respond identically to a therapy, ignoring the genetic and epigenomic variations that define our individuality. With ever-improving differentiation and phenotyping methods, patient-specific iPSC-derived cardiovascular cells allow unprecedented opportunities to discover new drug targets and screen compounds for cardiovascular disease. Imbued with the genetic information of an individual, iPSCs will vastly improve our ability to test drugs efficiently, as well as tailor and titrate drug therapy for each patient.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> David T Paik, Mark Chandy, Joseph C Wu<br><b>Journal:</b> Pharmacol. Rev.<br><b>ISSN:</b> 1521-0081<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.22<br><b>Two-year IF:</b> 19.51<br><b>SJR:</b> 7.422<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31871214">Link</a></b><br><br><b>Title:</b> A CRISPR-Cas9 System for Genome Editing of Fusarium proliferatum.<br><b>Abstract:</b> Fusarium proliferatum causes diverse diseases of many economically important plants. The fungus produces several mycotoxins of which the fumonisins are the most toxic. Currently, deletion of key genes for mycotoxin biosynthesis is a laborious and time-consuming procedure. We developed a novel CRISPR/Cas9-based genome-editing tool for the direct delivery of preassembled Cas9 ribonucleoproteins into protoplasts of F. proliferatum. Our CRISPR-Cas9 system couples a site-specific double-strand DNA break mediated by two Cas9 ribonucleoproteins with microhomology recombination requiring only 50-bp regions flanking the target gene. This system reduces the risk of off-target mutations and minimizes the risk of altering any gene adjacent to the target region. We used this tool to delete a polyketide synthase gene (FUM1) required for fumonisin biosynthesis. The mutants generated are no longer able to produce fumonisins, confirming the key role of FUM1 in fumonisin biosynthesis. Our CRISPR-Cas9 system is an important new tool for genetic studies of Fusarium.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Massimo Ferrara, Miriam Haidukowski, Antonio F Logrieco, John F Leslie, Giuseppina Mulè<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.22<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882627">Link</a></b><br><br><b>Title:</b> Ex Vivo/In vivo Gene Editing in Hepatocytes Using "All-in-One" CRISPR-Adeno-Associated Virus Vectors with a Self-Linearizing Repair Template.<br><b>Abstract:</b> Adeno-associated virus (AAV)-based vectors are considered efficient and safe gene delivery systems in gene therapy. We combined two guide RNA genes, Cas9, and a self-linearizing repair template in one vector (AIO-SL) to correct fumarylacetoacetate hydrolase (FAH) deficiency in mice. The vector genome of 5.73 kb was packaged into VP2-depleted AAV particles (AAV2/8<br><b>Publication date:</b> 2019-12-30<br><b>Authors:</b> Simon Alexander Krooss, Zhen Dai, Florian Schmidt, Alice Rovai, Julia Fakhiri, Akshay Dhingra, Qinggong Yuan, Taihua Yang, Asha Balakrishnan, Lars Steinbrück, Sangar Srivaratharajan, Michael Peter Manns, Axel Schambach, Dirk Grimm, Jens Bohne, Amar Deep Sharma, Hildegard Büning, Michael Ott<br><b>Journal:</b> iScience<br><b>ISSN:</b> 2589-0042<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.22<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31887661">Link</a></b><br><br><b>Title:</b> Genetic tool for fate mapping of Oct4 (Pou5f1)-expressing cells and their progeny past the pluripotency stage.<br><b>Abstract:</b> BACKGROUND: Methods based on site-specific recombinases are widely used in studying gene activities in vivo and in vitro. In these studies, constitutively active or inducible variants of these recombinases are expressed under the control of either lineage-specific or ubiquitous promoters. However, there is a need for more advanced schemes that combine these features with possibilities to choose a time point from which lineage tracing starts in an autonomous fashion. For example, the key mammalian germline gatekeeper gene Oct4 (Pou5f1) is expressed in the peri-implantation epiblast which gives rise to all cells within embryos. Thus the above techniques are hardly applicable to Oct4 tracing past the epiblast stage, and the establishment of genetic tools addressing such a limitation is a highly relevant pursuit.
METHODS: The CRISPR/Cas9 tool was used to manipulate the genome of mouse embryonic stem cells (ESCs), and various cell culture technics-to maintain and differentiate ESCs to neural cell, lentivirus-based reprogramming technique-to generate induced pluripotent stem cells (iPSCs).
RESULTS: In this paper, we have developed a two-component genetic system (referred to as O4S) that allows tracing Oct4 gene activity past the epiblast stage of development. The first component represents a knock-in of an ubiquitous promoter-driven inducible Cre, serving as a stop signal for downstream tdTomato. Upon activation of Cre activity with 4-hydroxytamoxifen (4-OHT) at any given time point, the recombinase excises a stop signal and poses the second component of the system-the FlpO recombinase, knocked into 3'UTR of Oct4, to be expressed upon activation of the latter gene. Oct4-driven expression of FlpO, in turn, triggers the tdTomato expression and thus, permanently marks Oct4
CONCLUSIONS: The developed O4S system can be used to detect Oct4 activation event, both permanent and transient, in somatic cell types outside the germline. The approach can be equally adjusted to other genes, provided the first component of the system is placed under transcriptional control of these genes, thus, making it a valuable tool for cell fate mapping in mice.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Andrey A Kuzmin, Veronika V Ermakova, Sergey A Sinenko, Sergey V Ponomartsev, Tatiana Y Starkova, Elena V Skvortsova, Olga Cherepanova, Alexey N Tomilin<br><b>Journal:</b> Stem Cell Res Ther<br><b>ISSN:</b> 1757-6512<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.21<br><b>Two-year IF:</b> 4.66<br><b>SJR:</b> 1.548<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31843009">Link</a></b><br><br><b>Title:</b> The stem cell-specific long noncoding RNA HOXA10-AS in the pathogenesis of KMT2A-rearranged leukemia.<br><b>Abstract:</b> HOX genes are highly conserved, and their precisely controlled expression is crucial for normal hematopoiesis. Accordingly, deregulation of HOX genes can cause leukemia. However, despite of intensive research on the coding HOX genes, the role of the numerous long noncoding RNAs (lncRNAs) within the HOX clusters during hematopoiesis and their contribution to leukemogenesis are incompletely understood. Here, we show that the lncRNA HOXA10-AS, located antisense to HOXA10 and mir-196b in the HOXA cluster, is highly expressed in hematopoietic stem cells (HSCs) as well as in KMT2A-rearranged and NPM1 mutated acute myeloid leukemias (AMLs). Using short hairpin RNA- and locked nucleic acid-conjugated chimeric antisense oligonucleotide (LNA-GapmeR)-mediated HOXA10-AS-knockdown and CRISPR/Cas9-mediated excision in vitro, we demonstrate that HOXA10-AS acts as an oncogene in KMT2A-rearranged AML. Moreover, HOXA10-AS knockdown severely impairs the leukemic growth of KMT2A-rearranged patient-derived xenografts in vivo, while high HOXA10-AS expression can serve as a marker of poor prognosis in AML patients. Lentiviral expression of HOXA10-AS blocks normal monocytic differentiation of human CD34+ hematopoietic stem and progenitor cells. Mechanistically, we show that HOXA10-AS localizes in the cytoplasm and acts in trans to induce NF-κB target genes. In total, our data imply that the normally HSC-specific HOXA10-AS is an oncogenic lncRNA in KMT2A-r AML. Thus, it may also represent a potential therapeutic target in KMT2A-rearranged AML.<br><b>Publication date:</b> 2019-12-23<br><b>Authors:</b> Sina Al-Kershi, Raj Bhayadia, Michelle Ng, Lonneke Verboon, Stephan Emmrich, Lucie Gack, Adrian Schwarzer, Till Strowig, Dirk Heckl, Jan-Henning Klusmann<br><b>Journal:</b> Blood Adv<br><b>ISSN:</b> 2473-9537<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.19<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31867596">Link</a></b><br><br><b>Title:</b> CRISPR-Cas9 Causes Chromosomal Instability and Rearrangements in Cancer Cell Lines, Detectable by Cytogenetic Methods.<br><b>Abstract:</b> CRISPR-Cas9 has quickly become the method of choice for genome editing, with multiple publications describing technical advances and novel applications. It has been widely adopted as a tool for basic research and has significant translational and clinical potential. However, its usage has outpaced the establishment of essential and rigorous controls for unwanted off-target effects, manifested as small mutations, large deletions of target loci, or large-scale chromosomal rearrangements. A common application of CRISPR-Cas9 is as a tool for creating isogenic cell-line models to study the effects of precise mutations, or variants, on disease traits. Here, we describe the effect of standard CRISPR-Cas9 mutagenesis protocols on well characterized cancer cell lines. We demonstrate that commonly used methods for detecting correctly mutated clones fail to uncover large-scale rearrangements. We show that simple cytogenetic methods can be used to identify clones carrying chromosomal abnormalities and large mutations at target loci. These methods are quick and cost-efficient, and we suggest that such controls should be performed prior to publication of studies based on novel CRISPR-Cas9 mutated cancer cell lines.<br><b>Publication date:</b> 2019-11-19<br><b>Authors:</b> Emily Rayner, Mary-Anne Durin, Rachael Thomas, Daniela Moralli, Sean M O'Cathail, Ian Tomlinson, Catherine M Green, Annabelle Lewis<br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.19<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31742432">Link</a></b><br><br><b>Title:</b> Multi-functional genome-wide CRISPR system for high throughput genotype-phenotype mapping.<br><b>Abstract:</b> Genome-scale engineering is an indispensable tool to understand genome functions due to our limited knowledge of cellular networks. Unfortunately, most existing methods for genome-wide genotype-phenotype mapping are limited to a single mode of genomic alteration, i.e. overexpression, repression, or deletion. Here we report a multi-functional genome-wide CRISPR (MAGIC) system to precisely control the expression level of defined genes to desired levels throughout the whole genome. By combining the tri-functional CRISPR system and array-synthesized oligo pools, MAGIC is used to create, to the best of our knowledge, one of the most comprehensive and diversified genomic libraries in yeast ever reported. The power of MAGIC is demonstrated by the identification of previously uncharacterized genetic determinants of complex phenotypes, particularly those having synergistic interactions when perturbed to different expression levels. MAGIC represents a powerful synthetic biology tool to investigate fundamental biological questions as well as engineer complex phenotypes for biotechnological applications.<br><b>Publication date:</b> 2019-12-20<br><b>Authors:</b> Jiazhang Lian, Carl Schultz, Mingfeng Cao, Mohammad HamediRad, Huimin Zhao<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.17<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31857575">Link</a></b><br><br><b>Title:</b> Circulating tumor cells in precision oncology: clinical applications in liquid biopsy and 3D organoid model.<br><b>Abstract:</b> Circulating tumor cells (CTCs) are a rare subset of cells found in the blood of patients with solid tumors, which function as a seed for metastases. Cancer cells metastasize through the bloodstream either as single migratory CTCs or as multicellular groupings-CTC clusters. The CTCs preserve primary tumor heterogeneity and mimic tumor properties, and may be considered as clinical biomarker, preclinical model, and therapeutic target. The potential clinical application of CTCs is being a component of liquid biopsy. CTCs are also good candidates for generating preclinical models, especially 3D organoid cultures, which could be applied in drug screening, disease modeling, genome editing, tumor immunity, and organoid biobanks. In this review, we summarize current knowledge on the value and promise of evolving CTC technologies and highlight cutting-edge research on CTCs in liquid biopsy, tumor metastasis, and organoid preclinical models. The study of CTCs offers broad pathways to develop new biomarkers for tumor patient diagnosis, prognosis, and response to therapy, as well as translational models accelerating oncologic drug development.<br><b>Publication date:</b> 2019-12-23<br><b>Authors:</b> Chang Yang, Bai-Rong Xia, Wei-Lin Jin, Ge Lou<br><b>Journal:</b> Cancer Cell Int.<br><b>ISSN:</b> 1475-2867<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.17<br><b>Two-year IF:</b> 3.63<br><b>SJR:</b> 1.180<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31866766">Link</a></b><br><br><b>Title:</b> Vectofusin-1 Improves Transduction of Primary Human Cells with Diverse Retroviral and Lentiviral Pseudotypes, Enabling Robust, Automated Closed-System Manufacturing.<br><b>Abstract:</b> Cell and gene therapies are finally becoming viable patient treatment options, with both T cell- and hematopoietic stem cell (HSC)-based therapies being approved to market in Europe. However, these therapies, which involve the use of viral vector to modify the target cells, are expensive and there is an urgent need to reduce manufacturing costs. One major cost factor is the viral vector production itself, therefore improving the gene modification efficiency could significantly reduce the amount of vector required per patient. This study describes the use of a transduction enhancing peptide, Vectofusin-1<br><b>Publication date:</b> 2019-10-03<br><b>Authors:</b> Constanze Radek, Ornellie Bernadin, Katharina Drechsel, Nicole Cordes, Rita Pfeifer, Pia Sträßer, Mirella Mormin, Alejandra Gutierrez-Guerrero, François-Loïc Cosset, Andrew D Kaiser, Thomas Schaser, Anne Galy, Els Verhoeyen, Ian C D Johnston<br><b>Journal:</b> Hum. Gene Ther.<br><b>ISSN:</b> 1557-7422<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.17<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31578886">Link</a></b><br><br><b>Title:</b> Conditional control of RNA-guided nucleic acid cleavage and gene editing.<br><b>Abstract:</b> Prokaryotes use repetitive genomic elements termed CRISPR (clustered regularly interspaced short palindromic repeats) to destroy invading genetic molecules. Although CRISPR systems have been widely used in DNA and RNA technology, certain adverse effects do occur. For example, constitutively active CRISPR systems may lead to a certain risk of off-target effects. Here, we introduce post-synthetic masking and chemical activation of guide RNA (gRNA) to controlling CRISPR systems. An RNA structure profiling probe (2-azidomethylnicotinic acid imidazolide) is used. Moreover, we accomplish conditional control of gene editing in live cells. This proof-of-concept study demonstrates promising potential of chemical activation of gRNAs as a versatile tool for chemical biology.<br><b>Publication date:</b> 2020-01-04<br><b>Authors:</b> Shao-Ru Wang, Ling-Yu Wu, Hai-Yan Huang, Wei Xiong, Jian Liu, Lai Wei, Ping Yin, Tian Tian, Xiang Zhou<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.16<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31900392">Link</a></b><br><br><b>Title:</b> Mathematical modeling of self-contained CRISPR gene drive reversal systems.<br><b>Abstract:</b> There is a critical need for further research into methods to control biological populations. Numerous challenges to agriculture, ecological systems, and human health could be mitigated by the targeted reduction and management of key species (e.g. pests, parasites, and vectors for pathogens). The discovery and adaptation of the CRISPR/Cas editing platform co-opted from bacteria has provided a mechanism for a means to alter an entire population. A CRISPR-based gene drive system can allow for the forced propagation of a genetic element that bypasses Mendelian inheritance which can be used to bias sex determination, install exogenous information, or remove endogenous DNA within an entire species. Laboratory studies have demonstrated the potency by which gene drives can operate within insects and other organisms. However, continued research and eventual application face serious opposition regarding issues of policy, biosafety, effectiveness, and reversal. Previous mathematical work has suggested the use of modified gene drive designs that are limited in spread such as daisy chain or underdominance drives. However, no system has yet been proposed that allows for an inducible reversal mechanism without requiring the introduction of additional individuals. Here, we study gene drive effectiveness, fitness, and inducible drive systems that could respond to external stimuli expanding from a previous frequency-based population model. We find that programmed modification during gene drive propagation could serve as a potent safeguard to either slow or completely reverse drive systems and allow for a return to the original wild-type population.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Matthew G Heffel, Gregory C Finnigan<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.16<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882576">Link</a></b><br><br><b>Title:</b> Efficient production and transmission of CRISPR/Cas9-mediated mutant alleles at the IG-DMR via generation of mosaic mice using a modified 2CC method.<br><b>Abstract:</b> Generation of mutant imprinting control region (ICR) mice using genome editing is an important approach for elucidating ICR functions. IG-DMR is an ICR in the Dlk1-Dio3 imprinted domain that contains functional regions-in both parental alleles-that are essential for embryonic development. One drawback of this approach is that embryonic lethality can occur from aberrant expression of the imprinted genes if IG-DMR gets mutated in either the paternal or maternal allele. To overcome this problem, we generated mosaic mice that contained cells with modified IG-DMR alleles and wild-type cells using the 2CC method that allowed for microinjection of the CRISPR/Cas9 constructs into a blastomere of 2-cell embryos. This method improved the birth rate of the founder pups relative to that obtained using the standard protocol. We also successfully produced mosaic mice in which the tandem repeat array sequence in the IG-DMR had been replaced by homology directed repair. Additionally, paternal transmission of the replaced allele caused aberrant expression of the imprinted genes due to hypomethylation of the IG-DMR, indicating that the replaced allele recapitulated our deletion model. Our results indicate that this method is useful for the generation of mutant mice in which a genomic locus essential for normal development has been genetically edited.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Satoshi Hara, Miho Terao, Akari Muramatsu, Shuji Takada<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.16<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882978">Link</a></b><br><br><b>Title:</b> Safety of Germline Genome Editing for Genetically Related "Future" Children as Perceived by Parents.<br><b>Abstract:</b> The social acceptability of germline genome editing (GGE) depends on its perceived safety, as well as respect for reproductive autonomy. However, it is doubtful that prospective parents sufficiently understand the risks of GGE. In the future, the use of GGE in specific situations seems plausible, as it offers couples potential means to safeguard genetically related future children from a serious disease and overcome infertility due to a gene mutation. Should GGE fail, however, some couples may be obliged to abort affected fetuses, or give birth to adversely affected children, which would be a tragedy. Some children might develop diseases later in life due to overlooked off-target mutations. Compounding this, some parents are unlikely to inform their offspring about the details of conception, hampering necessary follow-up. Prospective parents, scientists and policy makers should carefully discuss the safety implications of GGE for genetically related future children.<br><b>Publication date:</b> 2019-11-20<br><b>Authors:</b> Tetsuya Ishii, Iñigo de Miguel Beriain<br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.16<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31746634">Link</a></b><br><br><b>Title:</b> Agreement between two large pan-cancer CRISPR-Cas9 gene dependency data sets.<br><b>Abstract:</b> Genome-scale CRISPR-Cas9 viability screens performed in cancer cell lines provide a systematic approach to identify cancer dependencies and new therapeutic targets. As multiple large-scale screens become available, a formal assessment of the reproducibility of these experiments becomes necessary. We analyze data from recently published pan-cancer CRISPR-Cas9 screens performed at the Broad and Sanger Institutes. Despite significant differences in experimental protocols and reagents, we find that the screen results are highly concordant across multiple metrics with both common and specific dependencies jointly identified across the two studies. Furthermore, robust biomarkers of gene dependency found in one data set are recovered in the other. Through further analysis and replication experiments at each institute, we show that batch effects are driven principally by two key experimental parameters: the reagent library and the assay length. These results indicate that the Broad and Sanger CRISPR-Cas9 viability screens yield robust and reproducible findings.<br><b>Publication date:</b> 2019-12-21<br><b>Authors:</b> Joshua M Dempster, Clare Pacini, Sasha Pantel, Fiona M Behan, Thomas Green, John Krill-Burger, Charlotte M Beaver, Scott T Younger, Victor Zhivich, Hanna Najgebauer, Felicity Allen, Emanuel Gonçalves, Rebecca Shepherd, John G Doench, Kosuke Yusa, Francisca Vazquez, Leopold Parts, Jesse S Boehm, Todd R Golub, William C Hahn, David E Root, Mathew J Garnett, Aviad Tsherniak, Francesco Iorio<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31862961">Link</a></b><br><br><b>Title:</b> Targeting T cell malignancies using CAR-based immunotherapy: challenges and potential solutions.<br><b>Abstract:</b> Chimeric antigen receptor (CAR) T cell therapy has been successful in treating B cell malignancies in clinical trials; however, fewer studies have evaluated CAR T cell therapy for the treatment of T cell malignancies. There are many challenges in translating this therapy for T cell disease, including fratricide, T cell aplasia, and product contamination. To the best of our knowledge, no tumor-specific antigen has been identified with universal expression on cancerous T cells, hindering CAR T cell therapy for these malignancies. Numerous approaches have been assessed to address each of these challenges, such as (i) disrupting target antigen expression on CAR-modified T cells, (ii) targeting antigens with limited expression on T cells, and (iii) using third party donor cells that are either non-alloreactive or have been genome edited at the T cell receptor α constant (TRAC) locus. In this review, we discuss CAR approaches that have been explored both in preclinical and clinical studies targeting T cell antigens, as well as examine other potential strategies that can be used to successfully translate this therapy for T cell disease.<br><b>Publication date:</b> 2019-12-30<br><b>Authors:</b> Lauren C Fleischer, H Trent Spencer, Sunil S Raikar<br><b>Journal:</b> J Hematol Oncol<br><b>ISSN:</b> 1756-8722<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 8.71<br><b>SJR:</b> 2.960<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31884955">Link</a></b><br><br><b>Title:</b> Cysteine catabolism and the serine biosynthesis pathway support pyruvate production during pyruvate kinase knockdown in pancreatic cancer cells.<br><b>Abstract:</b> Background: Pancreatic ductal adenocarcinoma (PDAC) is an aggressive cancer with limited treatment options. Pyruvate kinase, especially the M2 isoform (PKM2), is highly expressed in PDAC cells, but its role in pancreatic cancer remains controversial. To investigate the role of pyruvate kinase in pancreatic cancer, we knocked down PKM2 individually as well as both PKM1 and PKM2 concurrently (PKM1/2) in cell lines derived from a 
Methods: We used liquid chromatography tandem mass spectrometry (LC-MS/MS) to determine metabolic profiles of wildtype and PKM1/2 knockdown PDAC cells. We further used stable isotope-labeled metabolic precursors and LC-MS/MS to determine metabolic pathways upregulated in PKM1/2 knockdown cells. We then targeted metabolic pathways upregulated in PKM1/2 knockdown cells using CRISPR/Cas9 gene editing technology.
Results: PDAC cells are able to proliferate and continue to produce pyruvate despite PKM1/2 knockdown. The serine biosynthesis pathway partially contributed to pyruvate production during PKM1/2 knockdown: knockout of phosphoglycerate dehydrogenase in this pathway decreased pyruvate production from glucose. In addition, cysteine catabolism generated ~ 20% of intracellular pyruvate in PDAC cells. Other potential sources of pyruvate include the sialic acid pathway and catabolism of glutamine, serine, tryptophan, and threonine. However, these sources did not provide significant levels of pyruvate in PKM1/2 knockdown cells.
Conclusion: PKM1/2 knockdown does not impact the proliferation of pancreatic cancer cells. The serine biosynthesis pathway supports conversion of glucose to pyruvate during pyruvate kinase knockdown. However, direct conversion of serine to pyruvate was not observed during PKM1/2 knockdown. Investigating several alternative sources of pyruvate identified cysteine catabolism for pyruvate production during PKM1/2 knockdown. Surprisingly, we find that a large percentage of intracellular pyruvate comes from cysteine. Our results highlight the ability of PDAC cells to adaptively rewire their metabolic pathways during knockdown of a key metabolic enzyme.<br><b>Publication date:</b> 2020-01-01<br><b>Authors:</b> Lei Yu, Shao Thing Teoh, Elliot Ensink, Martin P Ogrodzinski, Che Yang, Ana I Vazquez, Sophia Y Lunt<br><b>Journal:</b> Cancer Metab<br><b>ISSN:</b> 2049-3002<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 6.14<br><b>SJR:</b> 3.928<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31893043">Link</a></b><br><br><b>Title:</b> A genome-scale CRISPR/Cas9 knockout screening reveals SH3D21 as a sensitizer for gemcitabine.<br><b>Abstract:</b> Gemcitabine, 2',2'-difluoro-2'-deoxycytidine, is used as a pro-drug in treatment of variety of solid tumour cancers including pancreatic cancer. After intake, gemcitabine is transferred to the cells by the membrane nucleoside transporter proteins. Once inside the cells, it is converted to gemcitabine triphosphate followed by incorporation into DNA chains where it causes inhibition of DNA replication and thereby cell cycle arrest and apoptosis. Currently gemcitabine is the standard drug for treatment of pancreatic cancer and despite its widespread use its effect is moderate. In this study, we performed a genome-scale CRISPR/Cas9 knockout screening on pancreatic cancer cell line Panc1 to explore the genes that are important for gemcitabine efficacy. We found SH3D21 as a novel gemcitabine sensitizer implying it may act as a therapeutic target for improvement of gemcitabine efficacy in treatment of pancreatic cancer.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Mohammad Masoudi, Motoaki Seki, Razieh Yazdanparast, Nozomu Yachie, Hiroyuki Aburatani<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31844142">Link</a></b><br><br><b>Title:</b> A ligand-based system for receptor-specific delivery of proteins.<br><b>Abstract:</b> Gene delivery using vector or viral-based methods is often limited by technical and safety barriers. A promising alternative that circumvents these shortcomings is the direct delivery of proteins into cells. Here we introduce a non-viral, ligand-mediated protein delivery system capable of selectively targeting primary skin cells in-vivo. Using orthologous self-labelling tags and chemical cross-linkers, we conjugate large proteins to ligands that bind their natural receptors on the surface of keratinocytes. Targeted CRE-mediated recombination was achieved by delivery of ligand cross-linked CRE protein to the skin of transgenic reporter mice, but was absent in mice lacking the ligand's cell surface receptor. We further show that ligands mediate the intracellular delivery of Cas9 allowing for CRISPR-mediated gene editing in the skin more efficiently than adeno-associated viral gene delivery. Thus, a ligand-based system enables the effective and receptor-specific delivery of large proteins and may be applied to the treatment of skin-related genetic diseases.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Mariano Maffei, Chiara Morelli, Ellie Graham, Stefano Patriarca, Laura Donzelli, Balint Doleschall, Fernanda de Castro Reis, Linda Nocchi, Cora H Chadick, Luc Reymond, Ivan R Corrêa, Kai Johnsson, Jamie A Hackett, Paul A Heppenstall<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31844114">Link</a></b><br><br><b>Title:</b> CRISPR/Cas9-mediated targeted mutagenesis of GmLHY genes alters plant height and internode length in soybean.<br><b>Abstract:</b> BACKGROUND: Soybean (Glycine max) is an economically important oil and protein crop. Plant height is a key trait that significantly impacts the yield of soybean; however, research on the molecular mechanisms associated with soybean plant height is lacking. The CRISPR (clustered regularly interspaced short palindromic repeat)/Cas9 (CRISPR-associated system 9) system is a recently developed technology for gene editing that has been utilized to edit the genomes of crop plants.
RESULTS: Here, we designed four gRNAs to mutate four LATE ELONGATED HYPOCOTYL (LHY) genes in soybean. In order to test whether the gRNAs could perform properly in transgenic soybean plants, we first tested the CRISPR construct in transgenic soybean hairy roots using Agrobacterium rhizogenes strain K599. Once confirmed, we performed stable soybean transformation and obtained 19 independent transgenic soybean plants. Subsequently, we obtained one T
CONCLUSIONS: Our results indicate that CRISPR/Cas9-mediated targeted mutagenesis of four GmLHY genes reduces soybean plant height and shortens internodes from 20 to 35 days after emergence (DAE). These findings provide insight into the mechanisms underlying plant height regulatory networks in soybean.<br><b>Publication date:</b> 2019-12-19<br><b>Authors:</b> Qun Cheng, Lidong Dong, Tong Su, Tingyu Li, Zhuoran Gan, Haiyang Nan, Sijia Lu, Chao Fang, Lingping Kong, Haiyang Li, Zhihong Hou, Kun Kou, Yang Tang, Xiaoya Lin, Xiaohui Zhao, Liyu Chen, Baohui Liu, Fanjiang Kong<br><b>Journal:</b> BMC Plant Biol.<br><b>ISSN:</b> 1471-2229<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 3.83<br><b>SJR:</b> 1.687<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31852439">Link</a></b><br><br><b>Title:</b> An Introduction to CRISPR-Mediated Genome Editing in Fungi.<br><b>Abstract:</b> Central dogma, transformation, and genome editing are key biological concepts for which junior scientists must gain experience during training. Here we present an exercise that introduces these concepts in a single practical laboratory exercise. Our exercise utilizes CRISPR/Cas9 genome editing to introduce a stop codon into <br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Valmik K Vyas, Douglas A Bernstein<br><b>Journal:</b> J Microbiol Biol Educ<br><b>ISSN:</b> 1935-7877<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.14<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31890080">Link</a></b><br><br><b>Title:</b> A bacterial gene-drive system efficiently edits and inactivates a high copy number antibiotic resistance locus.<br><b>Abstract:</b> Gene-drive systems in diploid organisms bias the inheritance of one allele over another. CRISPR-based gene-drive expresses a guide RNA (gRNA) into the genome at the site where the gRNA directs Cas9-mediated cleavage. In the presence of Cas9, the gRNA cassette and any linked cargo sequences are copied via homology-directed repair (HDR) onto the homologous chromosome. Here, we develop an analogous CRISPR-based gene-drive system for the bacterium Escherichia coli that efficiently copies a gRNA cassette and adjacent cargo flanked with sequences homologous to the targeted gRNA/Cas9 cleavage site. This "pro-active" genetic system (Pro-AG) functionally inactivates an antibiotic resistance marker on a high copy number plasmid with ~ 100-fold greater efficiency than control CRISPR-based methods, suggesting an amplifying positive feedback loop due to increasing gRNA dosage. Pro-AG can likewise effectively edit large plasmids or single-copy genomic targets or introduce functional genes, foreshadowing potential applications to biotechnology or biomedicine.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> J Andrés Valderrama, Surashree S Kulkarni, Victor Nizet, Ethan Bier<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31844051">Link</a></b><br><br><b>Title:</b> High-frequency random DNA insertions upon co-delivery of CRISPR-Cas9 ribonucleoprotein and selectable marker plasmid in rice.<br><b>Abstract:</b> An important advantage of delivering CRISPR reagents into cells as a ribonucleoprotein (RNP) complex is the ability to edit genes without reagents being integrated into the genome. Transient presence of RNP molecules in cells can reduce undesirable off-target effects. One method for RNP delivery into plant cells is the use of a biolistic gun. To facilitate selection of transformed cells during RNP delivery, a plasmid carrying a selectable marker gene can be co-delivered with the RNP to enrich for transformed/edited cells. In this work, we compare targeted mutagenesis in rice using three different delivery platforms: biolistic RNP/DNA co-delivery; biolistic DNA delivery; and Agrobacterium-mediated delivery. All three platforms were successful in generating desired mutations at the target sites. However, we observed a high frequency (over 14%) of random plasmid or chromosomal DNA fragment insertion at the target sites in transgenic events generated from both biolistic delivery platforms. In contrast, integration of random DNA fragments was not observed in transgenic events generated from the Agrobacterium-mediated method. These data reveal important insights that must be considered when selecting the method for genome-editing reagent delivery in plants, and emphasize the importance of employing appropriate molecular screening methods to detect unintended alterations following genome engineering.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Raviraj Banakar, Alan L Eggenberger, Keunsub Lee, David A Wright, Karthik Murugan, Scott Zarecor, Carolyn J Lawrence-Dill, Dipali G Sashital, Kan Wang<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882637">Link</a></b><br><br><b>Title:</b> R534C mutation in hERG causes a trafficking defect in iPSC-derived cardiomyocytes from patients with type 2 long QT syndrome.<br><b>Abstract:</b> Patient-specific cardiomyocytes obtained from induced pluripotent stem cells (CM-iPSC) offer unprecedented mechanistic insights in the study of inherited cardiac diseases. The objective of this work was to study a type 2 long QT syndrome (LQTS2)-associated mutation (c.1600C > T in KCNH2, p.R534C in hERG) in CM-iPSC. Peripheral blood mononuclear cells were isolated from two patients with the R534C mutation and iPSCs were generated. In addition, the same mutation was inserted in a control iPSC line by genome editing using CRISPR/Cas9. Cells expressed pluripotency markers and showed spontaneous differentiation into the three embryonic germ layers. Electrophysiology demonstrated that action potential duration (APD) of LQTS2 CM-iPSC was significantly longer than that of the control line, as well as the triangulation of the action potentials (AP), implying a longer duration of phase 3. Treatment with the I<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Fernanda C P Mesquita, Paulo C Arantes, Tais H Kasai-Brunswick, Dayana S Araujo, Fernanda Gubert, Gustavo Monnerat, Danúbia Silva Dos Santos, Gabriel Neiman, Isabela C Leitão, Raiana A Q Barbosa, Jorge L Coutinho, Isadora M Vaz, Marcus N Dos Santos, Tamara Borgonovo, Fernando E S Cruz, Santiago Miriuka, Emiliano H Medei, Antonio C Campos de Carvalho, Adriana B Carvalho<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31844156">Link</a></b><br><br><b>Title:</b> HIV-1 Rev interacts with HERV-K RcREs present in the human genome and promotes export of unspliced HERV-K proviral RNA.<br><b>Abstract:</b> BACKGROUND: The HERV-K (HML-2) viruses are the youngest of the human endogenous retroviruses. They are present as several almost complete proviral copies and numerous fragments in the human genome. Many HERV-K proviruses express a regulatory protein Rec, which binds to an element present in HERV-K mRNAs called the RcRE. This interaction is necessary for the nucleo-cytoplasmic export and expression of HERV-K mRNAs that retain introns and plays a role analogous to that of Rev and the RRE in HIV replication. There are over 900 HERV-K RcREs distributed throughout the human genome. Thus, it was of interest to determine if Rev could functionally interact with selected RcRE elements that map either to HERV-K proviruses or human gene regions. This interaction would have the potential to alter the expression of both HERV-K mRNAs and cellular mRNAs during HIV-1 infection.
RESULTS: In this study we employed a combination of RNAseq, bioinformatics and cell-based functional assays. Potential RcREs were identified through a number of bioinformatic approaches. They were then tested for their ability to promote export and translation of a reporter mRNA with a retained intron in conjunction with Rev or Rec. Some of the selected elements functioned well with either Rev, Rec or both, whereas some showed little or no function. Rev function on individual RcREs varied and was also dependent on the Rev sequence. We also performed RNAseq on total and cytoplasmic RNA isolated from SupT1 cells expressing HIV Rev, with or without Tat, or HERV-K Rec. Proviral mRNA from three HERV-K loci (4p16.1b, 22q11.23 and most significantly 3q12.3) accumulated in the cytoplasm in the presence of Rev or Tat and Rev, but not Rec. Consistent with this, the 3' RcRE from 3q12.3 functioned well with HIV-Rev in our reporter assay. In contrast, this RcRE showed little or no function with Rec.
CONCLUSIONS: The HIV Rev protein can functionally interact with many RcREs present in the human genome, depending on the RcRE sequence, as well as the Rev sequence. This leads to export of some of the HERV-K proviral mRNAs and also has the potential to change the expression of non-viral genes.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Laurie R Gray, Rachel E Jackson, Patrick E H Jackson, Stefan Bekiranov, David Rekosh, Marie-Louise Hammarskjöld<br><b>Journal:</b> Retrovirology<br><b>ISSN:</b> 1742-4690<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 3.49<br><b>SJR:</b> 1.738<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31842941">Link</a></b><br><br><b>Title:</b> In vivo and in vitro knockout system labelled using fluorescent protein via microhomology-mediated end joining.<br><b>Abstract:</b> Gene knockout is important for understanding gene function and genetic disorders. The CRISPR/Cas9 system has great potential to achieve this purpose. However, we cannot distinguish visually whether a gene is knocked out and in how many cells it is knocked out among a population of cells. Here, we developed a new system that enables the labelling of knockout cells with fluorescent protein through microhomology-mediated end joining-based knock-in. Using a combination with recombinant adeno-associated virus, we delivered our system into the retina, where the expression of <br><b>Publication date:</b> 2019-12-25<br><b>Authors:</b> Shota Katayama, Kota Sato, Toru Nakazawa<br><b>Journal:</b> Life Sci Alliance<br><b>ISSN:</b> 2575-1077<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31874862">Link</a></b><br><br><b>Title:</b> Expanding the CRISPR Toolbox with ErCas12a in Zebrafish and Human Cells.<br><b>Abstract:</b> CRISPR and CRISPR-Cas effector proteins enable the targeting of DNA double-strand breaks to defined loci based on a variable length RNA guide specific to each effector. The guide RNAs are generally similar in size and form, consisting of a ∼20 nucleotide sequence complementary to the DNA target and an RNA secondary structure recognized by the effector. However, the effector proteins vary in protospacer adjacent motif requirements, nuclease activities, and DNA binding kinetics. Recently, ErCas12a, a new member of the Cas12a family, was identified in <br><b>Publication date:</b> 2019-11-19<br><b>Authors:</b> Wesley A Wierson, Brandon W Simone, Zachary WareJoncas, Carla Mann, Jordan M Welker, Bibekananda Kar, Michael J Emch, Iddo Friedberg, William A C Gendron, Michael A Barry, Karl J Clark, Drena L Dobbs, Maura A McGrail, Stephen C Ekker, Jeffrey J Essner<br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.13<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31742435">Link</a></b><br><br><b>Title:</b> Upgrading of efficient and scalable CRISPR-Cas-mediated technology for genetic engineering in thermophilic fungus <br><b>Abstract:</b> Background: Thermophilic filamentous fungus 
Results: In this study, we developed an 
Conclusions: Our study described new approaches (Camr technology) that allow easy and efficient marker recycling and iterative stacking of traits in the same thermophilic fungus strain either, using the newly established CRISPR-Cas12a system or the established CRISPR-Cas9 system. This Camr technology will be a versatile and efficient tool for engineering, theoretically, an unlimited number of genes in fungi. We expect this advance to accelerate biotechnology-oriented engineering processes in fungi.<br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Qian Liu, Yongli Zhang, Fangya Li, Jingen Li, Wenliang Sun, Chaoguang Tian<br><b>Journal:</b> Biotechnol Biofuels<br><b>ISSN:</b> 1754-6834<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.11<br><b>Two-year IF:</b> 5.62<br><b>SJR:</b> 1.762<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31890021">Link</a></b><br><br><b>Title:</b> Tumor suppressor properties of the small C-terminal domain phosphatases in non-small cell lung cancer.<br><b>Abstract:</b> Non-Small Cell Lung Cancer (NSCLC) is responsible for the majority of deaths caused by cancer. Small C-terminal domain (CTD) phosphatases (SCP), CTDSP1, CTDSP2 and CTDSPL (CTDSPs) belong to SCP/CTDSP subfamily and are involved in many vital cellular processes and tumorigenesis. High similarity of their structures suggests similar functions. However their role in NSCLC remains insufficiently understood. For the first time we revealed the suppressor function of CTDSPs leading to a significant growth slowdown and senescence of A549 lung adenocarcinoma (ADC) cells in vitro. Their tumor-suppressive activity can be realized through increasing the proportion of the active form of Rb protein dephosphorylated at Ser807/811, Ser780, and Ser795 (P<0.05) thereby negatively regulating cancer cell proliferation. Moreover, we observed that a frequent (84%, 39/46) and highly concordant (Spearman's rank correlation coefficient (rs) = 0.53-0.62, P≤0.01) down-regulation of CTDSPs and RB1 is characteristic of primary NSCLC samples (n=46). A clear difference in their mRNA levels was found between lung ADCs with and without lymph node metastases, but not in squamous cell carcinomas (SCCs) (P≤0.05). Based on The Cancer Genome Atlas (TCGA) data and the results obtained using the CrossHub tool, we suggest that the well-known oncogenic cluster miR-96/182/183 could be a common expression regulator of CTDSPs. Indeed, according to our qPCR, the expression of CTDSPs negatively correlates with these miRs, but positively correlates with their intronic miR-26a/b. Our results reflect functional association of CTDSP1, CTDSP2, and CTDSPL, expand knowledge about their suppressor properties through Rb dephosphorylation and provide new insights into the regulation of NSCLC growth.<br><b>Publication date:</b> 2019-11-27<br><b>Authors:</b> George S Krasnov, Grigory A Puzanov, Marina A Afanasyeva, Erdem B Dashinimaev, Khava S Vishnyakova, Artemy D Beniaminov, Alexei A Adzhubei, Tatiana T Kondratieva, Yegor E Yegorov, Vera N Senchenko<br><b>Journal:</b> Biosci. Rep.<br><b>ISSN:</b> 1573-4935<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.11<br><b>Two-year IF:</b> 2.51<br><b>SJR:</b> 1.028<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31774910">Link</a></b><br><br><b>Title:</b> Small molecules for mesenchymal stem cell fate determination.<br><b>Abstract:</b> Mesenchymal stem cells (MSCs) are adult stem cells harboring self-renewal and multilineage differentiation potential that are capable of differentiating into osteoblasts, adipocytes, or chondrocytes <br><b>Publication date:</b> 2019-12-26<br><b>Authors:</b> Yu-Hao Cheng, Jing-Cheng Dong, Qin Bian<br><b>Journal:</b> World J Stem Cells<br><b>ISSN:</b> 1948-0210<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.11<br><b>Two-year IF:</b> 2.45<br><b>SJR:</b> 0.543<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31875870">Link</a></b><br><br><b>Title:</b> PASTMUS: mapping functional elements at single amino acid resolution in human cells.<br><b>Abstract:</b> Identification of functional elements for a protein of interest is important for achieving a mechanistic understanding. However, it remains cumbersome to assess each and every amino acid of a given protein in relevance to its functional significance. Here, we report a strategy, PArsing fragmented DNA Sequences from CRISPR Tiling MUtagenesis Screening (PASTMUS), which provides a streamlined workflow and a bioinformatics pipeline to identify critical amino acids of proteins in their native biological contexts. Using this approach, we map six proteins-three bacterial toxin receptors and three cancer drug targets, and acquire their corresponding functional maps at amino acid resolution.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Xinyi Zhang, Di Yue, Yinan Wang, Yuexin Zhou, Ying Liu, Yeting Qiu, Feng Tian, Ying Yu, Zhuo Zhou, Wensheng Wei<br><b>Journal:</b> Genome Biol.<br><b>ISSN:</b> 1474-760X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 12.16<br><b>SJR:</b> 9.867<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31842968">Link</a></b><br><br><b>Title:</b> Curing hemophilia A by NHEJ-mediated ectopic F8 insertion in the mouse.<br><b>Abstract:</b> BACKGROUND: Hemophilia A, a bleeding disorder resulting from F8 mutations, can only be cured by gene therapy. A promising strategy is CRISPR-Cas9-mediated precise insertion of F8 in hepatocytes at highly expressed gene loci, such as albumin (Alb). Unfortunately, the precise in vivo integration efficiency of a long insert is very low (~ 0.1%).
RESULTS: We report that the use of a double-cut donor leads to a 10- to 20-fold increase in liver editing efficiency, thereby completely reconstituting serum F8 activity in a mouse model of hemophilia A after hydrodynamic injection of Cas9-sgAlb and B domain-deleted (BDD) F8 donor plasmids. We find that the integration of a double-cut donor at the Alb locus in mouse liver is mainly through non-homologous end joining (NHEJ)-mediated knock-in. We then target BDDF8 to multiple sites on introns 11 and 13 and find that NHEJ-mediated insertion of BDDF8 restores hemostasis. Finally, using 3 AAV8 vectors to deliver genome editing components, including Cas9, sgRNA, and BDDF8 donor, we observe the same therapeutic effects. A follow-up of 100 mice over 1 year shows no adverse effects.
CONCLUSIONS: These findings lay the foundation for curing hemophilia A by NHEJ knock-in of BDDF8 at Alb introns after AAV-mediated delivery of editing components.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Jian-Ping Zhang, Xin-Xin Cheng, Mei Zhao, Guo-Hua Li, Jing Xu, Feng Zhang, Meng-Di Yin, Fei-Ying Meng, Xin-Yue Dai, Ya-Wen Fu, Zhi-Xue Yang, Cameron Arakaki, Ruijun Jeanna Su, Wei Wen, Wen-Tian Wang, Wanqiu Chen, Hannah Choi, Charles Wang, Guangping Gao, Lei Zhang, Tao Cheng, Xiao-Bing Zhang<br><b>Journal:</b> Genome Biol.<br><b>ISSN:</b> 1474-760X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 12.16<br><b>SJR:</b> 9.867<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31843008">Link</a></b><br><br><b>Title:</b> Sertoli cell ablation and replacement of the spermatogonial niche in mouse.<br><b>Abstract:</b> Spermatogonia, which produce sperm throughout the male lifetime, are regulated inside a niche composed of Sertoli cells, and other testis cell types. Defects in Sertoli cells often lead to infertility, but replacement of defective cells has been limited by the inability to deplete the existing population. Here, we use an FDA-approved non-toxic drug, benzalkonium chloride (BC), to deplete testis cell types in vivo. Four days after BC administration, Sertoli cells are preferentially depleted, and can be replaced to promote spermatogenesis from surviving (host) spermatogonia. Seven days after BC treatment, multiple cell types can be engrafted from fresh or cryopreserved testicular cells, leading to complete spermatogenesis from donor cells. These methods will be valuable for investigation of niche-supporting cell interactions, have the potential to lead to a therapy for idiopathic male infertility in the clinic, and could open the door to production of sperm from other species in the mouse.<br><b>Publication date:</b> 2020-01-03<br><b>Authors:</b> Tetsuhiro Yokonishi, Jennifer McKey, Shintaro Ide, Blanche Capel<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31896751">Link</a></b><br><br><b>Title:</b> MyoD induced enhancer RNA interacts with hnRNPL to activate target gene transcription during myogenic differentiation.<br><b>Abstract:</b> Emerging evidence supports roles of enhancer RNAs (eRNAs) in regulating target gene. Here, we study eRNA regulation and function during skeletal myoblast differentiation. We provide a panoramic view of enhancer transcription and categorization of eRNAs. Master transcription factor MyoD is crucial in activating eRNA production. Super enhancer (se) generated seRNA-1 and -2 promote myogenic differentiation in vitro and in vivo. seRNA-1 regulates expression levels of two nearby genes, myoglobin (Mb) and apolipoprotein L6 (Apol6), by binding to heterogeneous nuclear ribonucleoprotein L (hnRNPL). A CAAA tract on seRNA-1 is essential in mediating seRNA-1/hnRNPL binding and function. Disruption of seRNA-1-hnRNPL interaction attenuates Pol II and H3K36me3 deposition at the Mb locus, in coincidence with the reduction of its transcription. Furthermore, analyses of hnRNPL binding transcriptome-wide reveal its association with eRNAs is a general phenomenon in multiple cells. Collectively, we propose that eRNA-hnRNPL interaction represents a mechanism contributing to target mRNA activation.<br><b>Publication date:</b> 2019-12-20<br><b>Authors:</b> Yu Zhao, Jiajian Zhou, Liangqiang He, Yuying Li, Jie Yuan, Kun Sun, Xiaona Chen, Xichen Bao, Miguel A Esteban, Hao Sun, Huating Wang<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31857580">Link</a></b><br><br><b>Title:</b> A solid-phase transfection platform for arrayed CRISPR screens.<br><b>Abstract:</b> Arrayed CRISPR-based screens emerge as a powerful alternative to pooled screens making it possible to investigate a wide range of cellular phenotypes that are typically not amenable to pooled screens. Here, we describe a solid-phase transfection platform that enables CRISPR-based genetic screens in arrayed format with flexible readouts. We demonstrate efficient gene knockout upon delivery of guide RNAs and Cas9/guide RNA ribonucleoprotein complexes into untransformed and cancer cell lines. In addition, we provide evidence that our platform can be easily adapted to high-throughput screens and we use this approach to study oncogene addiction in tumor cells. Finally demonstrating that the human primary cells can also be edited using this method, we pave the way for rapid testing of potential targeted therapies.<br><b>Publication date:</b> 2019-12-30<br><b>Authors:</b> Özdemirhan Serçin, Sabine Reither, Paris Roidos, Nadja Ballin, Spyridon Palikyras, Anna Baginska, Katrin Rein, Maria Llamazares, Aliaksandr Halavatyi, Hauke Winter, Thomas Muley, Renata Z Jurkowska, Amir Abdollahi, Frank T Zenke, Beate Neumann, Balca R Mardin<br><b>Journal:</b> Mol. Syst. Biol.<br><b>ISSN:</b> 1744-4292<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 9.58<br><b>SJR:</b> 7.040<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31885201">Link</a></b><br><br><b>Title:</b> Inducing Tumor Suppressive Microenvironments through Genome Edited CD47<br><b>Abstract:</b> Tumors can escape from the immune system by overexpressing CD47 and other checkpoint blockades. CD47 is expressed ubiquitously by all cells in the body, posing an obstacle for CD47 blocking treatments due to their systemic toxicity. We performed a study to determine how the tumor microenvironment changes after vaccination with genome edited CD47<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Subhadra Jayaraman Rukmini, Huanjing Bi, Puloma Sen, Benjamin Everhart, Sha Jin, Kaiming Ye<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882679">Link</a></b><br><br><b>Title:</b> Episomal vectors based on S/MAR and the β-globin Replicator, encoding a synthetic transcriptional activator, mediate efficient γ-globin activation in haematopoietic cells.<br><b>Abstract:</b> We report the development of episomal vectors for the specific γ-globin transcription activation in its native position by activator Zif-VP64, based on the Scaffold/Matrix Attachment Region (S/MAR) for episomal retention and the β-globin Replicator, the DNA replication-Initiation Region from the β-globin locus. Vector Zif-VP64-Ep1 containing transcription cassettes CMV- Zif-VP64 and CMV-eGFP-S/MAR transfected a)K562 cells; b)murine β-YAC bone marrow cells (BMC); c)human haematopoietic progenitor CD34<br><b>Publication date:</b> 2019-12-25<br><b>Authors:</b> Eleana F Stavrou, Emannuouil Simantirakis, Meletios Verras, Carlos Barbas, George Vassilopoulos, Kenneth R Peterson, Aglaia Athanassiadou<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31874995">Link</a></b><br><br><b>Title:</b> Assessing the therapeutic potential of Graptopetalum paraguayense on Alzheimer's disease using patient iPSC-derived neurons.<br><b>Abstract:</b> Alzheimer's disease (AD) is the most common type of dementia and also one of the leading causes of death worldwide. However, the underlying mechanisms remain unclear, and currently there is no drug treatment that can prevent or cure AD. Here, we have applied the advantages of using induced pluripotent stem cell (iPSC)-derived neurons (iNs) from AD patients, which are able to offer human-specific drug responsiveness, in order to evaluate therapeutic candidates for AD. Using approach involving an inducible neurogenin-2 transgene, we have established a robust and reproducible protocol for differentiating human iPSCs into glutamatergic neurons. The AD-iN cultures that result have mature phenotypic and physiological properties, together with AD-like biochemical features that include extracellular β-amyloid (Aβ) accumulation and Tau protein phosphorylation. By screening using a gene set enrichment analysis (GSEA) approach, Graptopetalum paraguayense (GP) has been identified as a potential therapeutic agent for AD from among a range of Chinese herbal medicines. We found that administration of a GP extract caused a significantly reduction in the AD-associated phenotypes of the iNs, including decreased levels of extracellular Aβ40 and Aβ42, as well as reduced Tau protein phosphorylation at positions Ser214 and Ser396. Additionally, the effect of GP was more prominent in AD-iNs compared to non-diseased controls. These findings provide valuable information that suggests moving extracts of GP toward drug development, either for treating AD or as a health supplement to prevent AD. Furthermore, our human iN-based platform promises to be a useful strategy when it is used for AD drug discovery.<br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Pei-Chun Wu, Ming-Ji Fann, Tu Thanh Tran, Shu-Cian Chen, Tania Devina, Irene Han-Juo Cheng, Cheng-Chang Lien, Lung-Sen Kao, Shuu-Jiun Wang, Jong-Ling Fuh, Tsai-Teng Tzeng, Chi-Ying Huang, Young-Ji Shiao, Yu-Hui Wong<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31848379">Link</a></b><br><br><b>Title:</b> A Scalable CURE Using a CRISPR/Cas9 Fluorescent Protein Knock-In Strategy in <br><b>Abstract:</b> Genome editing with CRISPR/Cas9 technology has advanced from the lab bench to clinical application with multiple trials underway. This article introduces a course-based undergraduate experience (CURE) combining CRISPR/Cas9 genome editing (using a modified two-plasmid system) and the animal model <br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Eric Hastie, Ryan Sellers, Bruno Valan, David R Sherwood<br><b>Journal:</b> J Microbiol Biol Educ<br><b>ISSN:</b> 1935-7877<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31890079">Link</a></b><br><br><b>Title:</b> An Engineered Cas-Transposon System for Programmable and Site-Directed DNA Transpositions.<br><b>Abstract:</b> Efficient site-directed insertion of heterologous DNA into a genome remains an outstanding challenge. Recombinases that can integrate kilobase-sized DNA constructs are difficult to reprogram to user-defined loci, while genomic insertion using CRISPR-Cas methods relies on inefficient host DNA repair machinery. Here, we describe a Cas-Transposon (CasTn) system for genomic insertions that uses a Himar1 transposase fused to a catalytically dead dCas9 nuclease to mediate programmable, site-directed transposition. Using cell-free <br><b>Publication date:</b> 2019-11-19<br><b>Authors:</b> Sway P Chen, Harris H Wang<br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.1<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31742433">Link</a></b><br><br><b>Title:</b> Synapse alterations precede neuronal damage and storage pathology in a human cerebral organoid model of CLN3-juvenile neuronal ceroid lipofuscinosis.<br><b>Abstract:</b> The juvenile form of neuronal ceroid Lipofuscinosis (JNCL) is the most common form within this group of rare lysosomal storage disorders, causing pediatric neurodegeneration. The genetic disorder, which is caused by recessive mutations affecting the CLN3 gene, features progressive vision loss, cognitive and motor decline and other psychiatric conditions, seizure episodes, leading to premature death. Animal models have traditionally aid the understanding of the disease mechanisms and pathology and are very relevant for biomarker research and therapeutic testing. Nevertheless, there is a need for establishing reliable and predictive human cellular models to study the disease. Since patient material, particularly from children, is scarce and difficult to obtain, we generated an engineered a CLN3-mutant isogenic human induced pluripotent stem cell (hiPSC) line carrying the c.1054C → T pathologic variant, using state of the art CRISPR/Cas9 technology. To prove the suitability of the isogenic pair to model JNCL, we screened for disease-specific phenotypes in non-neuronal two-dimensional cell culture models as well as in cerebral brain organoids. Our data demonstrates that the sole introduction of the pathogenic variant gives rise to classical hallmarks of JNCL in vitro. Additionally, we discovered an alteration of the splicing caused by this particular mutation. Next, we derived cerebral organoids and used them as a neurodevelopmental model to study the particular effects of the CLN3<br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Gemma Gomez-Giro, Jonathan Arias-Fuenzalida, Javier Jarazo, Dagmar Zeuschner, Muhammad Ali, Nina Possemis, Silvia Bolognin, Rashi Halder, Christian Jäger, Willemijn F E Kuper, Peter M van Hasselt, Holm Zaehres, Antonio Del Sol, Herman van der Putten, Hans R Schöler, Jens C Schwamborn<br><b>Journal:</b> Acta Neuropathol Commun<br><b>ISSN:</b> 2051-5960<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 5.93<br><b>SJR:</b> 3.279<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31888773">Link</a></b><br><br><b>Title:</b> Parallel Reaction Monitoring reveals structure-specific ceramide alterations in the zebrafish.<br><b>Abstract:</b> Extensive characterisations of the zebrafish genome and proteome have established a foundation for the use of the zebrafish as a model organism; however, characterisation of the zebrafish lipidome has not been as comprehensive. In an effort to expand current knowledge of the zebrafish sphingolipidome, a Parallel Reaction Monitoring (PRM)-based liquid chromatography-mass spectrometry (LC-MS) method was developed to comprehensively quantify zebrafish ceramides. Comparison between zebrafish and a human cell line demonstrated remarkable overlap in ceramide composition, but also revealed a surprising lack of most sphingadiene-containing ceramides in the zebrafish. PRM analysis of zebrafish embryogenesis identified developmental stage-specific ceramide changes based on long chain base (LCB) length. A CRISPR-Cas9-generated zebrafish model of Farber disease exhibited reduced size, early mortality, and severe ceramide accumulation where the amplitude of ceramide change depended on both acyl chain and LCB lengths. Our method adds an additional level of detail to current understanding of the zebrafish lipidome, and could aid in the elucidation of structure-function associations in the context of lipid-related diseases.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Tejia Zhang, Sunia A Trauger, Charles Vidoudez, Kim P Doane, Brock R Pluimer, Randall T Peterson<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882772">Link</a></b><br><br><b>Title:</b> Sideroflexin 4 affects Fe-S cluster biogenesis, iron metabolism, mitochondrial respiration and heme biosynthetic enzymes.<br><b>Abstract:</b> Sideroflexin4 (SFXN4) is a member of a family of nuclear-encoded mitochondrial proteins. Rare germline mutations in SFXN4 lead to phenotypic characteristics of mitochondrial disease including impaired mitochondrial respiration and hematopoetic abnormalities. We sought to explore the function of this protein. We show that knockout of SFXN4 has profound effects on Fe-S cluster formation. This in turn diminishes mitochondrial respiratory chain complexes and mitochondrial respiration and causes a shift to glycolytic metabolism. SFXN4 knockdown reduces the stability and activity of cellular Fe-S proteins, affects iron metabolism by influencing the cytosolic aconitase-IRP1 switch, redistributes iron from the cytosol to mitochondria, and impacts heme synthesis by reducing levels of ferrochelatase and inhibiting translation of ALAS2. We conclude that SFXN4 is essential for normal functioning of mitochondria, is necessary for Fe-S cluster biogenesis and iron homeostasis, and plays a critical role in mitochondrial respiration and synthesis of heme.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Bibbin T Paul, Lia Tesfay, C R Winkler, Frank M Torti, Suzy V Torti<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31873120">Link</a></b><br><br><b>Title:</b> Terminal neuron localization to the upper cortical plate is controlled by the transcription factor NEUROD2.<br><b>Abstract:</b> Excitatory neurons of the mammalian cerebral cortex are organized into six functional layers characterized by unique patterns of connectivity, as well as distinctive physiological and morphological properties. Cortical layers appear after a highly regulated migration process in which cells move from the deeper, proliferative zone toward the superficial layers. Importantly, defects in this radial migration process have been implicated in neurodevelopmental and psychiatric diseases. Here we report that during the final stages of migration, transcription factor Neurogenic Differentiation 2 (Neurod2) contributes to terminal cellular localization within the cortical plate. In mice, in utero knockdown of Neurod2 resulted in reduced numbers of neurons localized to the uppermost region of the developing cortex, also termed the primitive cortical zone. Our ChIP-Seq and RNA-Seq analyses of genes regulated by NEUROD2 in the developing cortex identified a number of key target genes with known roles in Reelin signaling, a critical regulator of neuronal migration. Our focused analysis of regulation of the Reln gene, encoding the extracellular ligand REELIN, uncovered NEUROD2 binding to conserved E-box elements in multiple introns. Furthermore, we demonstrate that knockdown of NEUROD2 in primary cortical neurons resulted in a strong increase in Reln gene expression at the mRNA level, as well as a slight upregulation at the protein level. These data reveal a new role for NEUROD2 during the late stages of neuronal migration, and our analysis of its genomic targets offers new genes with potential roles in cortical lamination.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Gizem Guzelsoy, Cansu Akkaya, Dila Atak, Cory D Dunn, Alkan Kabakcioglu, Nurhan Ozlu, Gulayse Ince-Dunn<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31873146">Link</a></b><br><br><b>Title:</b> Prophages in Lactobacillus reuteri Are Associated with Fitness Trade-Offs but Can Increase Competitiveness in the Gut Ecosystem.<br><b>Abstract:</b> The gut microbiota harbors a diverse phage population that is largely derived from lysogens, which are bacteria that contain dormant phages in their genome. While the diversity of phages in gut ecosystems is getting increasingly well characterized, knowledge is limited on how phages contribute to the evolution and ecology of their host bacteria. Here, we show that biologically active prophages are widely distributed in phylogenetically diverse strains of the gut symbiont <br><b>Publication date:</b> 2019-11-04<br><b>Authors:</b> Jee-Hwan Oh, Xiaoxi B Lin, Shenwei Zhang, Stephanie L Tollenaar, Mustafa Özçam, Case Dunphy, Jens Walter, Jan-Peter van Pijkeren<br><b>Journal:</b> Appl. Environ. Microbiol.<br><b>ISSN:</b> 1098-5336<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.96<br><b>SJR:</b> 1.663<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31676478">Link</a></b><br><br><b>Title:</b> Regulation of Apoptosis by Autophagy to Enhance Cancer Therapy.<br><b>Abstract:</b> In cancer therapy, a principle goal is to kill cancer cells while minimizing death of normal cells. Traditional cytotoxic therapies and the newer agents that target specific signaling proteins that are critical for cancer cell growth do this by activating a specific type of programmed cell death - apoptosis. However, it has been well established that cancer cells have varying levels of responses to apoptotic stimuli, with some being close to an "apoptotic threshold" and others being further away and that this ultimately determines whether cancer therapy is successful or not. In this review, we will highlight how the underlying mechanisms that control apoptosis thresholds relate to another important homeostatic process in cell survival and cell death, autophagy, and discuss recent evidence suggesting how inhibition of autophagy can enhance the action of anti-cancer drugs by modulating the apoptotic response.<br><b>Publication date:</b> 2019-12-23<br><b>Authors:</b> Kenneth D Tompkins, Andrew Thorburn<br><b>Journal:</b> Yale J Biol Med<br><b>ISSN:</b> 1551-4056<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.86<br><b>SJR:</b> 1.206<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31866785">Link</a></b><br><br><b>Title:</b> Challenges and Perspectives in Homology-Directed Gene Targeting in Monocot Plants.<br><b>Abstract:</b> Continuing crop domestication/redomestication and modification is a key determinant of the adaptation and fulfillment of the food requirements of an exploding global population under increasingly challenging conditions such as climate change and the reduction in arable lands. Monocotyledonous crops are not only responsible for approximately 70% of total global crop production, indicating their important roles in human life, but also the first crops to be challenged with the abovementioned hurdles; hence, monocot crops should be the first to be engineered and/or de novo domesticated/redomesticated. A long time has passed since the first green revolution; the world is again facing the challenge of feeding a predicted 9.7 billion people in 2050, since the decline in world hunger was reversed in 2015. One of the major lessons learned from the first green revolution is the importance of novel and advanced trait-carrying crop varieties that are ideally adapted to new agricultural practices. New plant breeding techniques (NPBTs), such as genome editing, could help us succeed in this mission to create novel and advanced crops. Considering the importance of NPBTs in crop genetic improvement, we attempt to summarize and discuss the latest progress with major approaches, such as site-directed mutagenesis using molecular scissors, base editors and especially homology-directed gene targeting (HGT), a very challenging but potentially highly precise genome modification approach in plants. We therefore suggest potential approaches for the improvement of practical HGT, focusing on monocots, and discuss a potential approach for the regulation of genome-edited products.<br><b>Publication date:</b> 2019-12-20<br><b>Authors:</b> Tien Van Vu, Yeon Woo Sung, Jihae Kim, Duong Thi Hai Doan, Mil Thi Tran, Jae-Yean Kim<br><b>Journal:</b> Rice (N Y)<br><b>ISSN:</b> 1939-8425<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.84<br><b>SJR:</b> 1.499<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31858277">Link</a></b><br><br><b>Title:</b> PIWI Proteins and piRNAs in the Nervous System.<br><b>Abstract:</b> PIWI Argonaute proteins and Piwi-interacting RNAs (piRNAs) are expressed in all animal species and play a critical role in cellular defense by inhibiting the activation of transposable elements in the germline. Recently, new evidence suggests that PIWI proteins and piRNAs also play important roles in various somatic tissues, including neurons. This review summarizes the neuronal functions of the PIWI-piRNA pathway in multiple animal species, including their involvement in axon regeneration, behavior, memory formation, and transgenerational epigenetic inheritance of adaptive memory. This review also discusses the consequences of dysregulation of neuronal PIWI-piRNA pathways in certain neurological disorders, including neurodevelopmental and neurodegenerative diseases. A full understanding of neuronal PIWI-piRNA pathways will ultimately provide novel insights into small RNA biology and could potentially provide precise targets for therapeutic applications.<br><b>Publication date:</b> 2019-12-16<br><b>Authors:</b> Kyung Won Kim<br><b>Journal:</b> Mol. Cells<br><b>ISSN:</b> 0219-1032<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.58<br><b>SJR:</b> 1.499<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31838836">Link</a></b><br><br><b>Title:</b> Full-length transcriptome sequences of Agropyron cristatum facilitate the prediction of putative genes for thousand-grain weight in a wheat-A. cristatum translocation line.<br><b>Abstract:</b> BACKGROUND: Agropyron cristatum (L.) Gaertn. (2n = 4x = 28; genomes PPPP) is a wild relative of common wheat (Triticum aestivum L.) and provides many desirable genetic resources for wheat improvement. However, there is still a lack of reference genome and transcriptome information for A. cristatum, which severely impedes functional and molecular breeding studies.
RESULTS: Single-molecule long-read sequencing technology from Pacific Biosciences (PacBio) was used to sequence full-length cDNA from a mixture of leaves, roots, stems and caryopses and constructed the first full-length transcriptome dataset of A. cristatum, which comprised 44,372 transcripts. As expected, the PacBio transcripts were generally longer and more complete than the transcripts assembled via the Illumina sequencing platform in previous studies. By analyzing RNA-Seq data, we identified tissue-enriched transcripts and assessed their GO term enrichment; the results indicated that tissue-enriched transcripts were enriched for particular molecular functions that varied by tissue. We identified 3398 novel and 1352 A. cristatum-specific transcripts compared with the wheat gene model set. To better apply this A. cristatum transcriptome, the A. cristatum transcripts were integrated with the wheat genome as a reference sequence to try to identify candidate A. cristatum transcripts associated with thousand-grain weight in a wheat-A. cristatum translocation line, Pubing 3035.
CONCLUSIONS: Full-length transcriptome sequences were used in our study. The present study not only provides comprehensive transcriptomic insights and information for A. cristatum but also proposes a new method for exploring the functional genes of wheat relatives under a wheat genetic background. The sequence data have been deposited in the NCBI under BioProject accession number PRJNA534411.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Shenghui Zhou, Jinpeng Zhang, Haiming Han, Jing Zhang, Huihui Ma, Zhi Zhang, Yuqing Lu, Weihua Liu, Xinming Yang, Xiuquan Li, Lihui Li<br><b>Journal:</b> BMC Genomics<br><b>ISSN:</b> 1471-2164<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.53<br><b>SJR:</b> 1.829<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31881839">Link</a></b><br><br><b>Title:</b> Improving CRISPR guide design with consensus approaches.<br><b>Abstract:</b> BACKGROUND: CRISPR-based systems are playing an important role in modern genome engineering. A large number of computational methods have been developed to assist in the identification of suitable guides. However, there is only limited overlap between the guides that each tool identifies. This can motivate further development, but also raises the question of whether it is possible to combine existing tools to improve guide design.
RESULTS: We considered nine leading guide design tools, and their output when tested using two sets of guides for which experimental validation data is available. We found that consensus approaches were able to outperform individual tools. The best performance (with a precision of up to 0.912) was obtained when combining four of the tools and accepting all guides selected by at least three of them.
CONCLUSIONS: These results can be used to improve CRISPR-based studies, but also to guide further tool development. However, they only provide a short-term solution as the time and computational resources required to run four tools may be impractical in certain applications.<br><b>Publication date:</b> 2019-12-25<br><b>Authors:</b> Jacob Bradford, Dimitri Perrin<br><b>Journal:</b> BMC Genomics<br><b>ISSN:</b> 1471-2164<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 3.53<br><b>SJR:</b> 1.829<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31874627">Link</a></b><br><br><b>Title:</b> Bioengineering hemophilia A-specific microvascular grafts for delivery of full-length factor VIII into the bloodstream.<br><b>Abstract:</b> Hemophilia A (HA) is a bleeding disorder caused by mutations in the F8 gene encoding coagulation factor VIII (FVIII). Current treatments are based on regular infusions of FVIII concentrates throughout a patient's life. Alternatively, viral gene therapies that directly deliver F8 in vivo have shown preliminary successes. However, hurdles remain, including lack of infection specificity and the inability to deliver the full-length version of F8 due to restricted viral cargo sizes. Here, we developed an alternative nonviral ex vivo gene-therapy approach that enables the overexpression of full-length F8 in patients' endothelial cells (ECs). We first generated HA patient-specific induced pluripotent stem cells (HA-iPSCs) from urine epithelial cells and genetically modified them using a piggyBac DNA transposon system to insert multiple copies of full-length F8. We subsequently differentiated the modified HA-iPSCs into competent ECs with high efficiency, and demonstrated that the cells (termed HA-FLF8-iECs) were capable of producing high levels of FVIII. Importantly, following subcutaneous implantation into immunodeficient hemophilic (SCID-f8ko) mice, we demonstrated that HA-FLF8-iECs were able to self-assemble into vascular networks, and that the newly formed microvessels had the capacity to deliver functional FVIII directly into the bloodstream of the mice, effectively correcting the clotting deficiency. Moreover, our implant maintains cellular confinement, which reduces potential safety concerns and allows effective monitoring and reversibility. We envision that this proof-of-concept study could become the basis for a novel autologous ex vivo gene-therapy approach to treat HA.<br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Joseph Neumeyer, Ruei-Zeng Lin, Kai Wang, Xuechong Hong, Tien Hua, Stacy E Croteau, Ellis J Neufeld, Juan M Melero-Martin<br><b>Journal:</b> Blood Adv<br><b>ISSN:</b> 2473-9537<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.08<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31851760">Link</a></b><br><br><b>Title:</b> Gastric squamous-columnar junction contains a large pool of cancer-prone immature osteopontin responsive Lgr5<br><b>Abstract:</b> Areas of a junction between two types of epithelia are known to be cancer-prone in many organ systems. However, mechanisms for preferential malignant transformation at the junction areas remain insufficiently elucidated. Here we report that inactivation of tumor suppressor genes Trp53 and Rb1 in the gastric squamous-columnar junction (SCJ) epithelium results in preferential formation of metastatic poorly differentiated neoplasms, which are similar to human gastroesophageal carcinoma. Unlike transformation-resistant antral cells, SCJ cells contain a highly proliferative pool of immature Lgr5<br><b>Publication date:</b> 2020-01-04<br><b>Authors:</b> Dah-Jiun Fu, Lianghai Wang, Fouad K Chouairi, Ian M Rose, Danysh A Abetov, Andrew D Miller, Robert J Yamulla, John C Schimenti, Andrea Flesken-Nikitin, Alexander Yu Nikitin<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31901081">Link</a></b><br><br><b>Title:</b> MLL-AF9 initiates transformation from fast-proliferating myeloid progenitors.<br><b>Abstract:</b> Cancer is a hyper-proliferative disease. Whether the proliferative state originates from the cell-of-origin or emerges later remains difficult to resolve. By tracking de novo transformation from normal hematopoietic progenitors expressing an acute myeloid leukemia (AML) oncogene MLL-AF9, we reveal that the cell cycle rate heterogeneity among granulocyte-macrophage progenitors (GMPs) determines their probability of transformation. A fast cell cycle intrinsic to these progenitors provide permissiveness for transformation, with the fastest cycling 3% GMPs acquiring malignancy with near certainty. Molecularly, we propose that MLL-AF9 preserves gene expression of the cellular states in which it is expressed. As such, when expressed in the naturally-existing, rapidly-cycling immature myeloid progenitors, this cell state becomes perpetuated, yielding malignancy. In humans, high CCND1 expression predicts worse prognosis for MLL fusion AMLs. Our work elucidates one of the earliest steps toward malignancy and suggests that modifying the cycling state of the cell-of-origin could be a preventative approach against malignancy.<br><b>Publication date:</b> 2019-12-19<br><b>Authors:</b> Xinyue Chen, Daniel B Burkhardt, Amaleah A Hartman, Xiao Hu, Anna E Eastman, Chao Sun, Xujun Wang, Mei Zhong, Smita Krishnaswamy, Shangqin Guo<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31852898">Link</a></b><br><br><b>Title:</b> Dissection of soybean populations according to selection signatures based on whole-genome sequences.<br><b>Abstract:</b> BACKGROUND: Domestication and improvement processes, accompanied by selections and adaptations, have generated genome-wide divergence and stratification in soybean populations. Simultaneously, soybean populations, which comprise diverse subpopulations, have developed their own adaptive characteristics enhancing fitness, resistance, agronomic traits, and morphological features. The genetic traits underlying these characteristics play a fundamental role in improving other soybean populations.
RESULTS: This study focused on identifying the selection signatures and adaptive characteristics in soybean populations. A core set of 245 accessions (112 wild-type, 79 landrace, and 54 improvement soybeans) selected from 4,234 soybean accessions was re-sequenced. Their genomic architectures were examined according to the domestication and improvement, and accessions were then classified into 3 wild-type, 2 landrace, and 2 improvement subgroups based on various population analyses. Selection and gene set enrichment analyses revealed that the landrace subgroups have selection signals for soybean-cyst nematode HG type 0 and seed development with germination, and that the improvement subgroups have selection signals for plant development with viability and seed development with embryo development, respectively. The adaptive characteristic for soybean-cyst nematode was partially underpinned by multiple resistance accessions, and the characteristics related to seed development were supported by our phenotypic findings for seed weights. Furthermore, their adaptive characteristics were also confirmed as genome-based evidence, and unique genomic regions that exhibit distinct selection and selective sweep patterns were revealed for 13 candidate genes.
CONCLUSIONS: Although our findings require further biological validation, they provide valuable information about soybean breeding strategies and present new options for breeders seeking donor lines to improve soybean populations.<br><b>Publication date:</b> 2019-12-23<br><b>Authors:</b> Jae-Yoon Kim, Seongmun Jeong, Kyoung Hyoun Kim, Won-Jun Lim, Ho-Yeon Lee, Namhee Jeong, Jung-Kyung Moon, Namshin Kim<br><b>Journal:</b> Gigascience<br><b>ISSN:</b> 2047-217X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 6.95<br><b>SJR:</b> 4.726<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31869408">Link</a></b><br><br><b>Title:</b> Mutations that improve efficiency of a weak-link enzyme are rare compared to adaptive mutations elsewhere in the genome.<br><b>Abstract:</b> New enzymes often evolve by gene amplification and divergence. Previous experimental studies have followed the evolutionary trajectory of an amplified gene, but have not considered mutations elsewhere in the genome when fitness is limited by an evolving gene. We have evolved a strain of <br><b>Publication date:</b> 2019-12-09<br><b>Authors:</b> Andrew B Morgenthaler, Wallis R Kinney, Christopher C Ebmeier, Corinne M Walsh, Daniel J Snyder, Vaughn S Cooper, William M Old, Shelley D Copley<br><b>Journal:</b> Elife<br><b>ISSN:</b> 2050-084X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 6.83<br><b>SJR:</b> 6.617<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31815667">Link</a></b><br><br><b>Title:</b> Distinct role of Sirtuin 1 (SIRT1) and Sirtuin 2 (SIRT2) in inhibiting cargo-loading and release of extracellular vesicles.<br><b>Abstract:</b> Exosomes, vehicles for intercellular communication, are formed intracellularly within multivesicular bodies (MVBs) and are released upon fusion with the plasma membrane. For their biogenesis, proper cargo loading to exosomes and vesicle traffic for extracellular release are required. Previously we showed that the L-type lectin, LMAN2, limits trans-Golgi Network (TGN)-to-endosomes traffic of GPRC5B, an exosome cargo protein, for exosome release. Here, we identified that the protein deacetylase sirtuin 2 (SIRT2) as a novel interactor of LMAN2. Loss of SIRT2 expression resulted in exosomal release of LMAN2, a Golgi resident protein, along with increased exosomal release of GPRC5B. Furthermore, knockout of SIRT2 increased total number of extracellular vesicles (EVs), indicating increased MVB-to-EV flux. While knockout of SIRT1 increased EV release with enlarged late endolysosome, knockout of SIRT2 did not exhibit endolysosome enlargement for increased EV release. Taken together, our study suggests that SIRT2 regulates cargo loading to MVBs and MVB-to-EV flux through a mechanism distinct from that of SIRT1.<br><b>Publication date:</b> 2019-12-28<br><b>Authors:</b> Byung Rho Lee, Bethany J Sanstrum, Yutao Liu, Sang-Ho Kwon<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31882861">Link</a></b><br><br><b>Title:</b> Injecting Gryllus bimaculatus Eggs.<br><b>Abstract:</b> Altering gene function in a developing organism is central to different kinds of experiments. While tremendously powerful genetic tools have been developed in traditional model systems, it is difficult to manipulate genes or messenger RNA (mRNA) in most other organisms. At the same time, evolutionary and comparative approaches rely on an exploration of gene function in many different species, necessitating the development and adaptation of techniques for manipulating expression outside currently genetically tractable species. This protocol describes a method for injecting reagents into cricket eggs to assay the effects of a given manipulation on embryonic or larval development. Instructions for how to collect and inject eggs with beveled needles are described. This relatively straightforward technique is flexible and potentially adaptable to other insects. One can gather and inject dozens of eggs in a single experiment, and survival rates for buffer-only injections improve with practice and can be as high as 80%. This technique will support several types of experimental approaches including injection of pharmacological agents, in vitro capped mRNA to express genes of interest, double-stranded RNA (dsRNA) to achieve RNA interference, use of clustered regularly interspaced short palindromic repeats (CRISPR) in concert with CRISPR-associated protein 9 (Cas9) reagents for genomic modification, and transposable elements to generate transient or stable transgenic lines.<br><b>Publication date:</b> 2019-09-09<br><b>Authors:</b> Samantha K Barry, Taro Nakamura, Yuji Matsuoka, Christoph Straub, Hadley W Horch, Cassandra G Extavour<br><b>Journal:</b> J Vis Exp<br><b>ISSN:</b> 1940-087X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 1.05<br><b>SJR:</b> 0.665<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31498320">Link</a></b><br><br><b>Title:</b> Origin of Lactose Fermentation in Kluyveromyces lactis by Interspecies Transfer of a Neo-functionalized Gene Cluster during Domestication.<br><b>Abstract:</b> Humans have used yeasts to make cheese and kefir for millennia, but the ability to ferment the milk sugar lactose is found in only a few yeast species, of which the foremost is Kluyveromyces lactis [1]. Two genes, LAC12 (lactose permease) and LAC4 (lactase), are sufficient for lactose uptake and hydrolysis to glucose and galactose [2]. Here, we show that these genes have a complex evolutionary history in the genus Kluyveromyces that is likely the result of human activity during domestication. We show that the ancestral Lac12 was bifunctional, able to import both lactose and cellobiose into the cell. These disaccharides were then hydrolyzed by Lac4 in the case of lactose or Cel2 in the case of cellobiose. A second cellobiose transporter, Cel1, was also present ancestrally. In the K. lactis lineage, the ancestral LAC12 and LAC4 were lost and a separate upheaval in the sister species K. marxianus resulted in loss of CEL1 and quadruplication of LAC12. One of these LAC12 genes became neofunctionalized to encode an efficient lactose transporter capable of supporting fermentation, specifically in dairy strains of K. marxianus, where it formed a LAC4-LAC12-CEL2 gene cluster, although another remained a cellobiose transporter. Then, the ability to ferment lactose was acquired very recently by K. lactis var. lactis by introgression of LAC12 and LAC4 on a 15-kb subtelomeric region from a dairy strain of K. marxianus. The genomic history of the LAC genes shows that strong selective pressures were imposed on yeasts by early dairy farmers.<br><b>Publication date:</b> 2019-12-09<br><b>Authors:</b> Javier A Varela, Martina Puricelli, Raúl A Ortiz-Merino, Romina Giacomobono, Stephanie Braun-Galleani, Kenneth H Wolfe, John P Morrissey<br><b>Journal:</b> Curr. Biol.<br><b>ISSN:</b> 1879-0445<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.06<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31813610">Link</a></b><br><br><b>Title:</b> Recommendations for application of the functional evidence PS3/BS3 criterion using the ACMG/AMP sequence variant interpretation framework.<br><b>Abstract:</b> BACKGROUND: The American College of Medical Genetics and Genomics (ACMG)/Association for Molecular Pathology (AMP) clinical variant interpretation guidelines established criteria for different types of evidence. This includes the strong evidence codes PS3 and BS3 for "well-established" functional assays demonstrating a variant has abnormal or normal gene/protein function, respectively. However, they did not provide detailed guidance on how functional evidence should be evaluated, and differences in the application of the PS3/BS3 codes are a contributor to variant interpretation discordance between laboratories. This recommendation seeks to provide a more structured approach to the assessment of functional assays for variant interpretation and guidance on the use of various levels of strength based on assay validation.
METHODS: The Clinical Genome Resource (ClinGen) Sequence Variant Interpretation (SVI) Working Group used curated functional evidence from ClinGen Variant Curation Expert Panel-developed rule specifications and expert opinions to refine the PS3/BS3 criteria over multiple in-person and virtual meetings. We estimated the odds of pathogenicity for assays using various numbers of variant controls to determine the minimum controls required to reach moderate level evidence. Feedback from the ClinGen Steering Committee and outside experts were incorporated into the recommendations at multiple stages of development.
RESULTS: The SVI Working Group developed recommendations for evaluators regarding the assessment of the clinical validity of functional data and a four-step provisional framework to determine the appropriate strength of evidence that can be applied in clinical variant interpretation. These steps are as follows: (1) define the disease mechanism, (2) evaluate the applicability of general classes of assays used in the field, (3) evaluate the validity of specific instances of assays, and (4) apply evidence to individual variant interpretation. We found that a minimum of 11 total pathogenic and benign variant controls are required to reach moderate-level evidence in the absence of rigorous statistical analysis.
CONCLUSIONS: The recommendations and approach to functional evidence evaluation described here should help clarify the clinical variant interpretation process for functional assays. Further, we hope that these recommendations will help develop productive partnerships with basic scientists who have developed functional assays that are useful for interrogating the function of a variety of genes.<br><b>Publication date:</b> 2020-01-01<br><b>Authors:</b> Sarah E Brnich, Ahmad N Abou Tayoun, Fergus J Couch, Garry R Cutting, Marc S Greenblatt, Christopher D Heinen, Dona M Kanavy, Xi Luo, Shannon M McNulty, Lea M Starita, Sean V Tavtigian, Matt W Wright, Steven M Harrison, Leslie G Biesecker, Jonathan S Berg, None None<br><b>Journal:</b> Genome Med<br><b>ISSN:</b> 1756-994X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.05<br><b>Two-year IF:</b> 10.47<br><b>SJR:</b> 5.084<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31892348">Link</a></b><br><br><b>Title:</b> Distinct IL-1α-responsive enhancers promote acute and coordinated changes in chromatin topology in a hierarchical manner.<br><b>Abstract:</b> How cytokine-driven changes in chromatin topology are converted into gene regulatory circuits during inflammation still remains unclear. Here, we show that interleukin (IL)-1α induces acute and widespread changes in chromatin accessibility via the TAK1 kinase and NF-κB at regions that are highly enriched for inflammatory disease-relevant SNPs. Two enhancers in the extended chemokine locus on human chromosome 4 regulate the IL-1α-inducible IL8 and CXCL1-3 genes. Both enhancers engage in dynamic spatial interactions with gene promoters in an IL-1α/TAK1-inducible manner. Microdeletions of p65-binding sites in either of the two enhancers impair NF-κB recruitment, suppress activation and biallelic transcription of the IL8/CXCL2 genes, and reshuffle higher-order chromatin interactions as judged by i4C interactome profiles. Notably, these findings support a dominant role of the IL8 "master" enhancer in the regulation of sustained IL-1α signaling, as well as for IL-8 and IL-6 secretion. CRISPR-guided transactivation of the IL8 locus or cross-TAD regulation by TNFα-responsive enhancers in a different model locus supports the existence of complex enhancer hierarchies in response to cytokine stimulation that prime and orchestrate proinflammatory chromatin responses downstream of NF-κB.<br><b>Publication date:</b> 2019-11-08<br><b>Authors:</b> Sinah-Sophia Weiterer, Johanna Meier-Soelch, Theodore Georgomanolis, Athanasia Mizi, Anna Beyerlein, Hendrik Weiser, Lilija Brant, Christin Mayr-Buro, Liane Jurida, Knut Beuerlein, Helmut Müller, Axel Weber, Ulas Tenekeci, Oliver Dittrich-Breiholz, Marek Bartkuhn, Andrea Nist, Thorsten Stiewe, Wilfred Fj van IJcken, Tabea Riedlinger, M Lienhard Schmitz, Argyris Papantonis, Michael Kracht<br><b>Journal:</b> EMBO J.<br><b>ISSN:</b> 1460-2075<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.05<br><b>Two-year IF:</b> 9.96<br><b>SJR:</b> 7.579<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31701553">Link</a></b><br><br><b>Title:</b> Current basis and future directions of zebrafish nutrigenomics.<br><b>Abstract:</b> This review investigates the current state of nutrigenomics in the zebrafish animal models. The zebrafish animal model has been used extensively in the study of disease onset and progression and associated molecular changes. In this review, we provide a synopsis of nutrigenomics using the zebrafish animal model. Obesity and dyslipidemia studies describe the genomics of dietary-induced obesity in relation to high-fat/high-calorie diets. Inflammation and cardiovascular studies describe dietary effects on the expression of acute inflammatory markers and resulting chronic inflammatory issues including atherosclerosis. We also evaluated the genomic response to bioactive dietary compounds associated with metabolic disorders. Carbohydrate metabolism and β-cell function studies describe the impacts of high-carbohydrate dietary challenges on nutritional programming. We also report tumorigenesis in relation to dietary carcinogen exposure studies that can result in permanent genomic changes. Vitamin and mineral deficiency studies demonstrate transgenerational genomic impacts of micronutrients in the diet and temporal expression changes. Circadian rhythm studies describe the relation between metabolism and natural temporal cycles of gene expression that impacts health. Bone formation studies describe the role of dietary composition that influences bone reabsorption regulation. Finally, this review provides future directions in the use of the zebrafish model for nutrigenomic and nutrigenetic research.<br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Michael B Williams, Stephen A Watts<br><b>Journal:</b> Genes Nutr<br><b>ISSN:</b> 1555-8932<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.05<br><b>Two-year IF:</b> 3.21<br><b>SJR:</b> 1.169<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31890052">Link</a></b><br><br><b>Title:</b> PROTACs: great opportunities for academia and industry.<br><b>Abstract:</b> Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem. Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied. Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety. As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems. PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions. PROTACs have been proven to show better selectivity compared to classic inhibitors. PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations. PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases. Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic. More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.<br><b>Publication date:</b> 2019-12-30<br><b>Authors:</b> Xiuyun Sun, Hongying Gao, Yiqing Yang, Ming He, Yue Wu, Yugang Song, Yan Tong, Yu Rao<br><b>Journal:</b> Signal Transduct Target Ther<br><b>ISSN:</b> 2059-3635<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.05<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31885879">Link</a></b><br><br><b>Title:</b> RNF208, an estrogen-inducible E3 ligase, targets soluble Vimentin to suppress metastasis in triple-negative breast cancers.<br><b>Abstract:</b> The development of triple-negative breast cancer (TNBC) negatively impacts both quality of life and survival in a high percentage of patients. Here, we show that RING finger protein 208 (RNF208) decreases the stability of soluble Vimentin protein through a polyubiquitin-mediated proteasomal degradation pathway, thereby suppressing metastasis of TNBC cells. RNF208 was significantly lower in TNBC than the luminal type, and low expression of RNF208 was strongly associated with poor clinical outcomes. Furthermore, RNF208 was induced by 17β-estradiol (E2) treatment in an estrogen receptor alpha (ΕRα)-dependent manner. Overexpression of RNF208 suppresses tumor formation and lung metastasis of TNBC cells. Mechanistically, RNF208 specifically polyubiquitinated the Lys97 residue within the head domain of Vimentin through interaction with the Ser39 residue of phosphorylated Vimentin, which exists as a soluble form, eventually facilitating proteasomal degradation of Vimentin. Collectively, our findings define RNF208 as a negative regulator of soluble Vimentin and a prognostic biomarker for TNBC cells.<br><b>Publication date:</b> 2019-12-21<br><b>Authors:</b> Kyoungwha Pang, Jinah Park, Sung Gwe Ahn, Jihee Lee, Yuna Park, Akira Ooshima, Seiya Mizuno, Satoshi Yamashita, Kyung-Soon Park, So-Young Lee, Joon Jeong, Toshikazu Ushijima, Kyung-Min Yang, Seong-Jin Kim<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31862882">Link</a></b><br><br><b>Title:</b> Recommendations for the collection and use of multiplexed functional data for clinical variant interpretation.<br><b>Abstract:</b> Variants of uncertain significance represent a massive challenge to medical genetics. Multiplexed functional assays, in which the functional effects of thousands of genomic variants are assessed simultaneously, are increasingly generating data that can be used as additional evidence for or against variant pathogenicity. Such assays have the potential to resolve variants of uncertain significance, thereby increasing the clinical utility of genomic testing. Existing standards from the American College of Medical Genetics and Genomics (ACMG)/Association for Molecular Pathology (AMP) and new guidelines from the Clinical Genome Resource (ClinGen) establish the role of functional data in variant interpretation, but do not address the specific challenges or advantages of using functional data derived from multiplexed assays. Here, we build on these existing guidelines to provide recommendations to experimentalists for the production and reporting of multiplexed functional data and to clinicians for the evaluation and use of such data. By following these recommendations, experimentalists can produce transparent, complete, and well-validated datasets that are primed for clinical uptake. Our recommendations to clinicians and diagnostic labs on how to evaluate the quality of multiplexed functional datasets, and how different datasets could be incorporated into the ACMG/AMP variant-interpretation framework, will hopefully clarify whether and how such data should be used. The recommendations that we provide are designed to enhance the quality and utility of multiplexed functional data, and to promote their judicious use.<br><b>Publication date:</b> 2019-12-21<br><b>Authors:</b> Hannah Gelman, Jennifer N Dines, Jonathan Berg, Alice H Berger, Sarah Brnich, Fuki M Hisama, Richard G James, Alan F Rubin, Jay Shendure, Brian Shirts, Douglas M Fowler, Lea M Starita, None None<br><b>Journal:</b> Genome Med<br><b>ISSN:</b> 1756-994X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 10.47<br><b>SJR:</b> 5.084<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31862013">Link</a></b><br><br><b>Title:</b> Alanyl-tRNA Synthetase Quality Control Prevents Global Dysregulation of the Escherichia coli Proteome.<br><b>Abstract:</b> Mechanisms have evolved to prevent errors in replication, transcription, and translation of genetic material, with translational errors occurring most frequently. Errors in protein synthesis can occur at two steps, during tRNA aminoacylation and ribosome decoding. Recent advances in protein mass spectrometry have indicated that previous reports of translational errors have potentially underestimated the frequency of these events, but also that the majority of translational errors occur during ribosomal decoding, suggesting that aminoacylation errors are evolutionarily less tolerated. Despite that interpretation, there is evidence that some aminoacylation errors may be regulated, and thus provide a benefit to the cell, while others are clearly detrimental. Here, we show that while it has been suggested that regulated Thr-to-Ser substitutions may be beneficial, there is a threshold beyond which these errors are detrimental. In contrast, we show that errors mediated by alanyl-tRNA synthetase (AlaRS) are not well tolerated and induce a global stress response that leads to gross perturbation of the <br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Paul Kelly, Nicholas Backes, Kyle Mohler, Christopher Buser, Arundhati Kavoor, Jesse Rinehart, Gregory Phillips, Michael Ibba<br><b>Journal:</b> mBio<br><b>ISSN:</b> 2150-7511<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 6.50<br><b>SJR:</b> 3.849<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31848288">Link</a></b><br><br><b>Title:</b> Genomewide and Enzymatic Analysis Reveals Efficient d-Galacturonic Acid Metabolism in the Basidiomycete Yeast Rhodosporidium toruloides.<br><b>Abstract:</b> Biorefining of renewable feedstocks is one of the most promising routes to replace fossil-based products. Since many common fermentation hosts, such as <br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Ryan J Protzko, Christina A Hach, Samuel T Coradetti, Magdalena A Hackhofer, Sonja Magosch, Nils Thieme, Gina M Geiselman, Adam P Arkin, Jeffrey M Skerker, John E Dueber, J Philipp Benz<br><b>Journal:</b> mSystems<br><b>ISSN:</b> 2379-5077<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 6.28<br><b>SJR:</b> 2.979<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31848309">Link</a></b><br><br><b>Title:</b> The salivary gland proteome of root-galling grape phylloxera (Daktulosphaira vitifoliae Fitch) feeding on Vitis spp.<br><b>Abstract:</b> The successful parasitisation of a plant by a phytophagous insect is dependent on the delivery of effector molecules into the host. Sedentary gall forming insects, such as grape phylloxera (Daktulosphaira vitifoliae Fitch, Phylloxeridae), secrete multiple effectors into host plant tissues that alter or modulate the cellular and molecular environment to the benefit of the insect. The identification and characterisation of effector proteins will provide insight into the host-phylloxera interaction specifically the gall-induction processes and potential mechanisms of plant resistance. Using proteomic mass spectrometry and in-silico secretory prediction, 420 putative effectors were determined from the salivary glands or the root-feeding D. vitifoliae larvae reared on Teleki 5C (V. berlandieri x V. riparia). Among them, 170 conserved effectors were shared between D. vitifoliae and fourteen phytophagous insect species. Quantitative RT-PCR analysis of five conserved effector candidates (protein disulfide-isomerase, peroxidoredoxin, peroxidase and a carboxypeptidase) revealed that their gene expression decreased, when larvae were starved for 24 h, supporting their assignment as effector molecules. The D. vitifoliae effectors identified here represent a functionally diverse group, comprising both conserved and unique proteins that provide new insight into the D. vitifoliae-Vitis spp. interaction and the potential mechanisms by which D. vitifoliae establishes the feeding site, suppresses plant defences and modulates nutrient uptake.<br><b>Publication date:</b> 2019-12-17<br><b>Authors:</b> Markus W Eitle, James C Carolan, Michaela Griesser, Astrid Forneck<br><b>Journal:</b> PLoS ONE<br><b>ISSN:</b> 1932-6203<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 2.87<br><b>SJR:</b> 1.100<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31846459">Link</a></b><br><br><b>Title:</b> The transcriptome analysis of the Arabidopsis thaliana in response to the Vibrio vulnificus by RNA-sequencing.<br><b>Abstract:</b> Because of the recent increase in the demand for fresh produce, contamination of raw food products has become an issue. Foodborne diseases are frequently caused by the infection of leguminous plants by human bacterial pathogens. Moreover, contamination by Vibrio cholerae, closely related with Vibrio vulnificus, has been reported in plants and vegetables. Here, we investigated the possibility of Vibrio vulnificus 96-11-17M, an opportunistic human pathogen, to infect and colonize Arabidopsis thaliana plants, resulting in typical disease symptoms at 5 and 7 days post-inoculation in vitro and in planta under artificial and favorable conditions, respectively. RNA-Seq analysis revealed 5,360, 4,204, 4,916 and 3,741 differentially expressed genes (DEGs) at 12, 24, 48 and 72 h post-inoculation, respectively, compared with the 0 h time point. Gene Ontology analysis revealed that these DEGs act in pathways responsive to chemical and hormone stimuli and plant defense. The expression of genes involved in salicylic acid (SA)-, jasmonic acid (JA)- and ethylene (ET)-dependent pathways was altered following V. vulnificus inoculation. Genetic analyses of Arabidopsis mutant lines verified that common pathogen-associated molecular pattern (PAMP) receptors perceive the V. vulnificus infection, thus activating JA and ET signaling pathways. Our data indicate that the human bacterial pathogen V. vulnificus 96-11-17M modulates defense-related genes and host defense machinery in Arabidopsis thaliana under favorable conditions.<br><b>Publication date:</b> 2019-12-16<br><b>Authors:</b> Yong-Soon Park, Seon-Kyu Kim, Seon-Young Kim, Kyung Mo Kim, Choong-Min Ryu<br><b>Journal:</b> PLoS ONE<br><b>ISSN:</b> 1932-6203<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 2.87<br><b>SJR:</b> 1.100<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31841567">Link</a></b><br><br><b>Title:</b> Disorders of FZ-CRD; insights towards FZ-CRD folding and therapeutic landscape.<br><b>Abstract:</b> The ER is hub for protein folding. Proteins that harbor a Frizzled cysteine-rich domain (FZ-CRD) possess 10 conserved cysteine motifs held by a unique disulfide bridge pattern which attains a correct fold in the ER. Little is known about implications of disease-causing missense mutations within FZ-CRD families. Mutations in FZ-CRD of Frizzled class receptor 4 (FZD4) and Muscle, skeletal, receptor tyrosine kinase (MuSK) and Receptor tyrosine kinase-like orphan receptor 2 (ROR2) cause Familial Exudative Vitreoretinopathy (FEVR), Congenital Myasthenic Syndrome (CMS), and Robinow Syndrome (RS) respectively. We highlight reported pathogenic inherited missense mutations in FZ-CRD of FZD4, MuSK and ROR2 which misfold, and traffic abnormally in the ER, with ER-associated degradation (ERAD) as a common pathogenic mechanism for disease. Our review shows that all studied FZ-CRD mutants of RS, FEVR and CMS result in misfolded proteins and/or partially misfolded proteins with an ERAD fate, thus we coin them as "disorders of FZ-CRD". Abnormal trafficking was demonstrated in 17 of 29 mutants studied; 16 mutants were within and/or surrounding the FZ-CRD with two mutants distant from FZ-CRD. These ER-retained mutants were improperly N-glycosylated confirming ER-localization. FZD4 and MuSK mutants were tagged with polyubiquitin chains confirming targeting for proteasomal degradation. Investigating the cellular and molecular mechanisms of these mutations is important since misfolded protein and ER-targeted therapies are in development. The P344R-MuSK kinase mutant showed around 50% of its in-vitro autophosphorylation activity and P344R-MuSK increased two-fold on proteasome inhibition. M105T-FZD4, C204Y-FZD4, and P344R-MuSK mutants are thermosensitive and therefore, might benefit from extending the investigation to a larger number of chemical chaperones and/or proteasome inhibitors. Nonetheless, FZ-CRD ER-lipidation it less characterized in the literature and recent structural data sheds light on the importance of lipidation in protein glycosylation, proper folding, and ER trafficking. Current treatment strategies in-place for the conformational disease landscape is highlighted. From this review, we envision that disorders of FZ-CRD might be receptive to therapies that target FZ-CRD misfolding, regulation of fatty acids, and/or ER therapies; thus paving the way for a newly explored paradigm to treat different diseases with common defects.<br><b>Publication date:</b> 2020-01-01<br><b>Authors:</b> Reham M Milhem, Bassam R Ali<br><b>Journal:</b> Mol. Med.<br><b>ISSN:</b> 1528-3658<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 2.67<br><b>SJR:</b> 1.403<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31892318">Link</a></b><br><br><b>Title:</b> Expression of extraocular <br><b>Abstract:</b> Background: Animals living in well-lit environments utilize optical stimuli for detecting visual information, regulating the homeostatic pacemaker, and controlling patterns of body pigmentation. In contrast, many subterranean animal species without optical stimuli have evolved regressed binocular eyes and body pigmentation. Interestingly, some fossorial and cave-dwelling animals with regressed eyes still respond to light. These light-dependent responses may be simply evolutionary residuals or they may be adaptive, where negative phototaxis provides avoidance of predator-rich surface environments. However, the relationship between these non-ocular light responses and the underlying light-sensing Opsin proteins has not been fully elucidated.
Methods: To highlight the potential functions of opsins in a blind subterranean animal, we used the Mexican cave tetra to investigate opsin gene expression in the eyes and several brain regions of both surface and cave-dwelling adults. We performed database surveys, expression analyses by quantitative reverse transcription PCR (RT-qPCR), and light-dependent locomotor activity analysis using pinealectomized fish, one of the high-opsin expressing organs of cavefish.
Results: Based on conservative criteria, we identified 33 opsin genes in the cavefish genome. Surveys of available RNAseq data found 26 of these expressed in the surface fish eye as compared to 24 expressed in cavefish extraocular tissues, 20 of which were expressed in the brain. RT-qPCR of 26 opsins in surface and cavefish eye and brain tissues showed the highest opsin-expressing tissue in cavefish was the pineal organ, which expressed exo-rhodopsin at 72.7% of the expression levels in surface fish pineal. However, a pinealectomy resulted in no change to the light-dependent locomotor activity in juvenile cavefish and surface fish. Therefore, we conclude that, after 20,000 or more years of evolution in darkness, cavefish light-dependent basal activity is regulated by a non-pineal extraocular organ.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Noah Simon, Suguru Fujita, Megan Porter, Masato Yoshizawa<br><b>Journal:</b> PeerJ<br><b>ISSN:</b> 2167-8359<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 2.34<br><b>SJR:</b> 1.037<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31871836">Link</a></b><br><br><b>Title:</b> Determination of Pathogenicity of <br><b>Abstract:</b> Objectives: Molecular diagnostic laboratories screen for mutations in disease-causing genes in order to confirm a clinical diagnosis. The classification of DNA variants as 'pathogenic' or 'likely pathogenic' mutations creates a workflow bottleneck, which becomes increasingly challenging as greater number of genes are screened. The classification challenge is also acute if there are conflicting reports regarding pathogenicity and differing classification criteria between laboratories. This study aimed to compare two procedures for the classification of variants in the 
Methods: This bioinformatic study was conducted at LabPLUS, Auckland, New Zealand, from February to June 2017. DNA was extracted from peripheral blood samples of 30 patients and gene library construction was carried out using a commercially available targeted panel for the 
Results: The results showed concordance between the two classification protocols for a panel of 30 
Conclusion: The ACMG/AMP-based guidelines were applied to a cohort of patients with <br><b>Publication date:</b> 2020-01-03<br><b>Authors:</b> Angela Brown, Mansour Zamanpoor, Donald R Love, Debra O Prosser<br><b>Journal:</b> Sultan Qaboos Univ Med J<br><b>ISSN:</b> 2075-0528<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 0.80<br><b>SJR:</b> 0.332<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31897316">Link</a></b><br><br><b>Title:</b> Bigram-PGK: phosphoglycerylation prediction using the technique of bigram probabilities of position specific scoring matrix.<br><b>Abstract:</b> BACKGROUND: The biological process known as post-translational modification (PTM) is a condition whereby proteomes are modified that affects normal cell biology, and hence the pathogenesis. A number of PTMs have been discovered in the recent years and lysine phosphoglycerylation is one of the fairly recent developments. Even with a large number of proteins being sequenced in the post-genomic era, the identification of phosphoglycerylation remains a big challenge due to factors such as cost, time consumption and inefficiency involved in the experimental efforts. To overcome this issue, computational techniques have emerged to accurately identify phosphoglycerylated lysine residues. However, the computational techniques proposed so far hold limitations to correctly predict this covalent modification.
RESULTS: We propose a new predictor in this paper called Bigram-PGK which uses evolutionary information of amino acids to try and predict phosphoglycerylated sites. The benchmark dataset which contains experimentally labelled sites is employed for this purpose and profile bigram occurrences is calculated from position specific scoring matrices of amino acids in the protein sequences. The statistical measures of this work, such as sensitivity, specificity, precision, accuracy, Mathews correlation coefficient and area under ROC curve have been reported to be 0.9642, 0.8973, 0.8253, 0.9193, 0.8330, 0.9306, respectively.
CONCLUSIONS: The proposed predictor, based on the feature of evolutionary information and support vector machine classifier, has shown great potential to effectively predict phosphoglycerylated and non-phosphoglycerylated lysine residues when compared against the existing predictors. The data and software of this work can be acquired from https://github.com/abelavit/Bigram-PGK.<br><b>Publication date:</b> 2019-12-20<br><b>Authors:</b> Abel Chandra, Alok Sharma, Abdollah Dehzangi, Daichi Shigemizu, Tatsuhiko Tsunoda<br><b>Journal:</b> BMC Mol Cell Biol<br><b>ISSN:</b> 2661-8850<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.03<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31856704">Link</a></b><br><br><b>Title:</b> Evolution and Diversity of Assembly-Line Polyketide Synthases.<br><b>Abstract:</b> Assembly-line polyketide synthases (PKSs) are among the most complex protein machineries known in nature, responsible for the biosynthesis of numerous compounds used in the clinic. Their present-day diversity is the result of an evolutionary path that has involved the emergence of a multimodular architecture and further diversification of assembly-line PKSs. In this review, we provide an overview of previous studies that investigated PKS evolution and propose a model that challenges the currently prevailing view that gene duplication has played a major role in the emergence of multimodularity. We also analyze the ensemble of orphan PKS clusters sequenced so far to evaluate how large the entire diversity of assembly-line PKS clusters and their chemical products could be. Finally, we examine the existing techniques to access the natural PKS diversity in natural and heterologous hosts and describe approaches to further expand this diversity through engineering.<br><b>Publication date:</b> 2019-12-16<br><b>Authors:</b> Aleksandra Nivina, Kai P Yuet, Jake Hsu, Chaitan Khosla<br><b>Journal:</b> Chem. Rev.<br><b>ISSN:</b> 1520-6890<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 52.85<br><b>SJR:</b> 22.157<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31838842">Link</a></b><br><br><b>Title:</b> A bifunctional ATPase drives tad pilus extension and retraction.<br><b>Abstract:</b> A widespread class of prokaryotic motors powered by secretion motor adenosine triphosphatases (ATPases) drives the dynamic extension and retraction of extracellular fibers, such as type IV pili (T4P). Among these, the tight adherence (tad) pili are critical for surface sensing and biofilm formation. As for most other motors belonging to this class, how tad pili retract despite lacking a dedicated retraction motor ATPase has remained a mystery. Here, we find that a bifunctional pilus motor ATPase, CpaF, drives both activities through adenosine 5'-triphosphate (ATP) hydrolysis. We show that mutations within CpaF result in a correlated reduction in the rates of extension and retraction that directly scales with decreased ATP hydrolysis and retraction force. Thus, a single motor ATPase drives the bidirectional processes of pilus fiber extension and retraction.<br><b>Publication date:</b> 2020-01-03<br><b>Authors:</b> Courtney K Ellison, Jingbo Kan, Jennifer L Chlebek, Katherine R Hummels, Gaёl Panis, Patrick H Viollier, Nicolas Biais, Ankur B Dalia, Yves V Brun<br><b>Journal:</b> Sci Adv<br><b>ISSN:</b> 2375-2548<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 12.53<br><b>SJR:</b> 6.267<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31897429">Link</a></b><br><br><b>Title:</b> MAP3Kinase-dependent SnRK2-kinase activation is required for abscisic acid signal transduction and rapid osmotic stress response.<br><b>Abstract:</b> Abiotic stresses, including drought and salinity, trigger a complex osmotic-stress and abscisic acid (ABA) signal transduction network. The core ABA signalling components are snf1-related protein kinase2s (SnRK2s), which are activated by ABA-triggered inhibition of type-2C protein-phosphatases (PP2Cs). SnRK2 kinases are also activated by a rapid, largely unknown, ABA-independent osmotic-stress signalling pathway. Here, through a combination of a redundancy-circumventing genetic screen and biochemical analyses, we have identified functionally-redundant MAPKK-kinases (M3Ks) that are necessary for activation of SnRK2 kinases. These M3Ks phosphorylate a specific SnRK2/OST1 site, which is indispensable for ABA-induced reactivation of PP2C-dephosphorylated SnRK2 kinases. ABA-triggered SnRK2 activation, transcription factor phosphorylation and SLAC1 activation require these M3Ks in vitro and in plants. M3K triple knock-out plants show reduced ABA sensitivity and strongly impaired rapid osmotic-stress-induced SnRK2 activation. These findings demonstrate that this M3K clade is required for ABA- and osmotic-stress-activation of SnRK2 kinases, enabling robust ABA and osmotic stress signal transduction.<br><b>Publication date:</b> 2020-01-03<br><b>Authors:</b> Yohei Takahashi, Jingbo Zhang, Po-Kai Hsu, Paulo H O Ceciliato, Li Zhang, Guillaume Dubeaux, Shintaro Munemasa, Chennan Ge, Yunde Zhao, Felix Hauser, Julian I Schroeder<br><b>Journal:</b> Nat Commun<br><b>ISSN:</b> 2041-1723<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 11.80<br><b>SJR:</b> 5.992<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31896774">Link</a></b><br><br><b>Title:</b> Experiments in macaque monkeys provide critical insights into age-associated changes in cognitive and sensory function.<br><b>Abstract:</b> The use of animal models in brain aging research has led to numerous fundamental insights into the neurobiological processes that underlie changes in brain function associated with normative aging. Macaque monkeys have become the predominant nonhuman primate model system in brain aging research due to their striking similarities to humans in their behavioral capacities, sensory processing abilities, and brain architecture. Recent public concern about nonhuman primate research has made it imperative to attempt to clearly articulate the potential benefits to human health that this model enables. The present review will highlight how nonhuman primates provide a critical bridge between experiments conducted in rodents and development of therapeutics for humans. Several studies discussed here exemplify how nonhuman primate research has enriched our understanding of cognitive and sensory decline in the aging brain, as well as how this work has been important for translating mechanistic implications derived from experiments conducted in rodents to human brain aging research.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Daniel T Gray, Carol A Barnes<br><b>Journal:</b> Proc. Natl. Acad. Sci. U.S.A.<br><b>ISSN:</b> 1091-6490<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 9.35<br><b>SJR:</b> 5.601<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31871147">Link</a></b><br><br><b>Title:</b> Size-dependent tissue-specific biological effects of core-shell structured Fe<br><b>Abstract:</b> BACKGROUND: Understanding the in vivo size-dependent pharmacokinetics and toxicity of nanoparticles is crucial to determine their successful development. Systematic studies on the size-dependent biological effects of nanoparticles not only help to unravel unknown toxicological mechanism but also contribute to the possible biological applications of nanomaterial.
METHODS: In this study, the biodistribution and the size-dependent biological effects of Fe
RESULTS: Our findings indicated that biodistribution and biological activities of Fe@Si-NPs demonstrated the obvious size-dependent and tissue-specific effects. Spleen and liver are the target tissues of Fe@Si-NPs, and 20 nm of Fe@Si-NPs showed a possible longer blood circulation time. Quantitative biochemical analysis showed that the alterations of lactate dehydrogenase (LDH) and uric acid (UA) were correlated to some extent with the sizes of Fe@Si-NPs. The untargeted metabolomic analyses of tissue metabolomes (kidney, liver, lung, and spleen) indicated that different sizes of Fe@Si-NPs were involved in the different biochemical mechanisms. LDH, formate, uric acid, and GSH related metabolites were suggested as sensitive indicators for the size-dependent toxic effects of Fe@Si-NPs. The findings from serum biochemical analysis and metabolomic analysis corroborate each other. Thus we proposed a toxicity hypothesis that size-dependent NAD depletion may occur in vivo in response to nanoparticle exposure. To our knowledge, this is the first report that links size-dependent biological effects of nanoparticles with in vivo NAD depletion in rats.
CONCLUSION: The integrated metabolomic approach is an effective tool to understand physiological responses to the size-specific properties of nanoparticles. Our results can provide a direction for the future biological applications of Fe@Si-NPs.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Jinquan Li, Zhongxue Yuan, Huili Liu, Jianghua Feng, Zhong Chen<br><b>Journal:</b> J Nanobiotechnology<br><b>ISSN:</b> 1477-3155<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 5.21<br><b>SJR:</b> 1.255<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31870377">Link</a></b><br><br><b>Title:</b> Recent advances of drug delivery nanocarriers in osteosarcoma treatment.<br><b>Abstract:</b> Osteosarcoma is the most common primary malignant bone tumor mainly occurred in children and adolescence, and chemotherapy is limited for the side effects and development of drug resistance. Advances in nanotechnology and knowledge of cancer biology have led to significant improvements in developing tumor-targeted drug delivery nanocarriers, and some have even entered clinically application. Delivery of chemotherapeutic agents by functionalized smart nanocarriers could protect the drugs from rapid clearance, prolong the circulating time, and increase the drug concentration at tumor sites, thus enhancing the therapeutic efficacy and reducing side effects. Various drug delivery nanocarriers have been designed and tested for osteosarcoma treatment, but most of them are still at experimental stage, and more further studies are needed before clinical application. In this present review, we briefly describe the types of commonly used nanocarriers in osteosarcoma treatment, and discuss the strategies for osteosarcoma-targeted delivery and controlled release of drugs. The application of nanoparticles in the management of metastatic osteosarcoma is also briefly discussed. The purpose of this article is to present an overview of recent progress of nanoscale drug delivery platforms in osteosarcoma, and inspire new ideas to develop more effective therapeutic options.<br><b>Publication date:</b> 2020-01-01<br><b>Authors:</b> Shang-Yu Wang, Hong-Zhi Hu, Xiang-Cheng Qing, Zhi-Cai Zhang, Zeng-Wu Shao<br><b>Journal:</b> J Cancer<br><b>ISSN:</b> 1837-9664<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 3.22<br><b>SJR:</b> 0.993<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31892974">Link</a></b><br><br><b>Title:</b> Functional mutation allele mining of plant architecture and yield-related agronomic traits and characterization of their effects in wheat.<br><b>Abstract:</b> BACKGROUND: Wheat mutant resources with phenotypic variation have been developed in recent years. These mutants might carry favorable mutation alleles, which have the potential to be utilized in the breeding process. Plant architecture and yield-related features are important agronomic traits for wheat breeders and mining favorable alleles of these traits will improve wheat characteristics.
RESULTS: Here we used 190 wheat phenotypic mutants as material and by analyzing their SNP variation and phenotypic data, mutation alleles for plant architecture and yield-related traits were identified, and the genetic effects of these alleles were evaluated. In total, 32 mutation alleles, including three pleiotropic alleles, significantly associated with agronomic traits were identified from the 190 wheat mutant lines. The SNPs were distributed on 12 chromosomes and were associated with plant height (PH), tiller number, flag leaf angle (FLA), thousand grain weight (TGW), and other yield-related traits. Further phenotypic analysis of multiple lines carrying the same mutant allele was performed to determine the effect of the allele on the traits of interest. PH-associated SNPs on chromosomes 2BL, 3BS, 3DL, and 5DL might show additive effects, reducing PH by 10.0 cm to 31.3 cm compared with wild type, which means that these alleles may be favorable for wheat improvement. Only unfavorable mutation alleles that reduced TGW and tiller number were identified. A region on chromosome 5DL with mutation alleles for PH and TGW contained several long ncRNAs, and their sequences shared more than 90% identity with cytokinin oxidase/dehydrogenase genes. Some of the mutation alleles we mined were colocalized with previously reported QTLs or genes while others were novel; these novel alleles could also result in phenotypic variation.
CONCLUSION: Our results demonstrate that favorable mutation alleles are present in mutant resources, and the region between 409.5 to 419.8 Mb on chromosome 5DL affects wheat plant height and thousand grain weight.<br><b>Publication date:</b> 2019-12-31<br><b>Authors:</b> Huijun Guo, Hongchun Xiong, Yongdun Xie, Linshu Zhao, Jiayu Gu, Shirong Zhao, Yuping Ding, Luxiang Liu<br><b>Journal:</b> BMC Genet.<br><b>ISSN:</b> 1471-2156<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 2.66<br><b>SJR:</b> 1.025<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31888475">Link</a></b><br><br><b>Title:</b> Predicting synthetic lethal interactions in human cancers using graph regularized self-representative matrix factorization.<br><b>Abstract:</b> BACKGROUND: Synthetic lethality has attracted a lot of attentions in cancer therapeutics due to its utility in identifying new anticancer drug targets. Identifying synthetic lethal (SL) interactions is the key step towards the exploration of synthetic lethality in cancer treatment. However, biological experiments are faced with many challenges when identifying synthetic lethal interactions. Thus, it is necessary to develop computational methods which could serve as useful complements to biological experiments.
RESULTS: In this paper, we propose a novel graph regularized self-representative matrix factorization (GRSMF) algorithm for synthetic lethal interaction prediction. GRSMF first learns the self-representations from the known SL interactions and further integrates the functional similarities among genes derived from Gene Ontology (GO). It can then effectively predict potential SL interactions by leveraging the information provided by known SL interactions and functional annotations of genes. Extensive experiments on the synthetic lethal interaction data downloaded from SynLethDB database demonstrate the superiority of our GRSMF in predicting potential synthetic lethal interactions, compared with other competing methods. Moreover, case studies of novel interactions are conducted in this paper for further evaluating the effectiveness of GRSMF in synthetic lethal interaction prediction.
CONCLUSIONS: In this paper, we demonstrate that by adaptively exploiting the self-representation of original SL interaction data, and utilizing functional similarities among genes to enhance the learning of self-representation matrix, our GRSMF could predict potential SL interactions more accurately than other state-of-the-art SL interaction prediction methods.<br><b>Publication date:</b> 2019-12-24<br><b>Authors:</b> Jiang Huang, Min Wu, Fan Lu, Le Ou-Yang, Zexuan Zhu<br><b>Journal:</b> BMC Bioinformatics<br><b>ISSN:</b> 1471-2105<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0.02<br><b>Two-year IF:</b> 2.61<br><b>SJR:</b> 1.374<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31870274">Link</a></b><br><br><b>Title:</b> The <br><b>Abstract:</b> Sphingolipids are membrane and bioactive lipids that are required for many aspects of normal mammalian development and physiology. However, the importance of the regulatory mechanisms that control sphingolipid levels in these processes is not well understood. The mammalian ORMDL proteins (ORMDL1, 2 and 3) mediate feedback inhibition of the de novo synthesis pathway of sphingolipids by inhibiting serine palmitoyl transferase in response to elevated ceramide levels. To understand the function of ORMDL proteins in vivo, we studied mouse knockouts (KOs) of the <br><b>Publication date:</b> 2019-12-27<br><b>Authors:</b> Benjamin A Clarke, Saurav Majumder, Hongling Zhu, Y Terry Lee, Mari Kono, Cuiling Li, Caroline Khanna, Hailey Blain, Ronit Schwartz, Vienna L Huso, Colleen Byrnes, Galina Tuymetova, Teresa M Dunn, Maria L Allende, Richard L Proia<br><b>Journal:</b> Elife<br><b>ISSN:</b> 2050-084X<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 6.83<br><b>SJR:</b> 6.617<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31880535">Link</a></b><br><br><b>Title:</b> Comparative Genomics for the Elucidation of Multidrug Resistance in Candida lusitaniae.<br><b>Abstract:</b> Multidrug resistance (MDR) has emerged in hospitals due to the use of several agents administered in combination or sequentially to the same individual. We reported earlier MDR in <br><b>Publication date:</b> 2019-12-25<br><b>Authors:</b> Abhilash Kannan, Sandra A Asner, Emilie Trachsel, Steve Kelly, Josie Parker, Dominique Sanglard<br><b>Journal:</b> mBio<br><b>ISSN:</b> 2150-7511<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 6.50<br><b>SJR:</b> 3.849<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31874914">Link</a></b><br><br><b>Title:</b> Diversity in Natural Transformation Frequencies and Regulation across <br><b>Abstract:</b> In <br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Chelsea A Simpson, Ram Podicheti, Douglas B Rusch, Ankur B Dalia, Julia C van Kessel<br><b>Journal:</b> mBio<br><b>ISSN:</b> 2150-7511<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 6.50<br><b>SJR:</b> 3.849<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31848285">Link</a></b><br><br><b>Title:</b> I8-arachnotocin-an arthropod-derived G protein-biased ligand of the human vasopressin V<br><b>Abstract:</b> The neuropeptides oxytocin (OT) and vasopressin (VP) and their G protein-coupled receptors OTR, V<br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> Leopold Duerrauer, Edin Muratspahić, Jasmin Gattringer, Peter Keov, Helen C Mendel, Kevin D G Pfleger, Markus Muttenthaler, Christian W Gruber<br><b>Journal:</b> Sci Rep<br><b>ISSN:</b> 2045-2322<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 4.12<br><b>SJR:</b> 1.414<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31848378">Link</a></b><br><br><b>Title:</b> Initial phospholipid-dependent Irgb6 targeting to <br><b>Abstract:</b> None<br><b>Publication date:</b> 2019-12-19<br><b>Authors:</b> Youngae Lee, Hiroshi Yamada, Ariel Pradipta, Ji Su Ma, Masaaki Okamoto, Hikaru Nagaoka, Eizo Takashima, Daron M Standley, Miwa Sasai, Kohji Takei, Masahiro Yamamoto<br><b>Journal:</b> Life Sci Alliance<br><b>ISSN:</b> 2575-1077<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31852733">Link</a></b><br><br><b>Title:</b> 75th Congress of the Italian Society of Pediatrics : Bologna, Italy. 29 May - 1 June 2019.<br><b>Abstract:</b> None<br><b>Publication date:</b> 2019-12-18<br><b>Authors:</b> <br><b>Journal:</b> Ital J Pediatr<br><b>ISSN:</b> 1824-7288<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31849329">Link</a></b><br><br><b>Title:</b> Correction to: <br><b>Abstract:</b> None<br><b>Publication date:</b> 2019-10-28<br><b>Authors:</b> <br><b>Journal:</b> CRISPR J<br><b>ISSN:</b> 2573-1602<br><b>Relevance score 1:</b> 0.14<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://ncbi.nlm.nih.gov/pubmed/31657624">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6943980/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941985/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941962/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941877/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938853/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938499/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938490/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927590/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6919427/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6929984/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6929632/">Link</a></b><br><br><b>Title:</b> Unknown - Retrieval Error<br><b>Abstract:</b> Unknown - Retrieval Error<br><b>Publication date:</b> 1980-01-01<br><b>Authors:</b> <br><b>Journal:</b> Unknown - Retrieval Error<br><b>ISSN:</b> Unknown - Retrieval Error<br><b>Relevance score 1:</b> 0<br><b>Relevance score 2:</b> 0<br><b>Two-year IF:</b> 0.0<br><b>SJR:</b> 0.0<br><b><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6925031/">Link</a></b><br><br><br><a name="companies"></a><h2>Companies</h2> <br><br><b>Company Name:</b> CRISPR Therapeutics<br><b>Overview:</b> CRISPR Therapeutics is a leading gene-editing company focused on the development of transformative medicines using its proprietary CRISPR/Cas9 gene-editing platform. CRISPR/Cas9 is a revolutionary technology that allows for precise, directed changes to genomic DNA. Our multi-disciplinary team of world-class researchers and drug developers is working to translate this technology into breakthrough human therapeutics in a number of serious diseases. Our lead programs in beta-thalassemia and sickle cell disease have advanced to IND/CTA-enabling studies with a CTA filing planned by the end of 2017, and we are advancing additional programs in ex vivo and in vivo disease areas. In addition to our fully-owned programs, our strategic collaborations with Bayer AG and Vertex Pharmaceuticals expand our portfolio and enable us with unique capabilities. Through our private financings, partnerships, and IPO we have raised >$400M to fund and accelerate our portfolio. We have licensed the foundational CRISPR/Cas9 patent estate for human therapeutic use from our scientific founder, Dr. Emmanuelle Charpentier, who co-invented the application of CRISPR/Cas9 for gene editing. Our company is headquartered in Zug, Switzerland with R&D operations in Cambridge, Massachusetts, USA and some business operations in London, United Kingdom.<br><b>Specialties:</b> Gene Editing, AAV, Hematology, and Immuno-Oncology<br><b><a href="https://www.linkedin.com/company/crispr-therapeutics/about/">LinkedIn Profile</a></b><br><b><a href="http://www.crisprtx.com">Website</a></b><br><br><b>Company Name:</b> Orchard Therapeutics<br><b>Overview:</b> Orchard Therapeutics is a leading global fully integrated commercial-stage company dedicated to transforming the lives of patients with rare diseases through innovative gene therapies.

Orchard’s portfolio of autologous ex vivo gene therapy programs has demonstrated sustained clinical benefit in over 150 patients across five disease areas. These programs include Strimvelis®, the first autologous ex vivo gene therapy approved by the EMA in 2016, 3 programs in advanced registrational studies in MLD (metachromatic leukodystrophy), WAS (Wiskott Aldrich syndrome) and ADA-SCID (adenosine deaminase severe combined immunodeficiency), 2 other clinical programs in X-CGD (X-linked chronic granulomatous disease) and beta-thalassemia, as well as an extensive preclinical pipeline.

The company is partnered with world-leading institutions in gene therapy, including University College London, Great Ormond Street Hospital, the University of Manchester and Central Manchester University Hospitals, the University of California Los Angeles and Boston Children’s Hospital, and Telethon Institute of Gene Therapy/Ospedale San Raffaele.

Orchard is a publicly traded company (NASDAQ: ORTX) with offices in the UK and the US, including London, San Francisco and Boston.<br><b>Specialties:</b> Stem cell technology, Gene therapy, Orphan diseases, and Paediatric diseases<br><b><a href="https://www.linkedin.com/company/orchard-therapeutics/about/">LinkedIn Profile</a></b><br><b><a href="http://www.orchard-tx.com">Website</a></b><br><br><b>Company Name:</b> Trucode Gene Repair, Inc.<br><b>Overview:</b> Trucode Gene Repair is advancing its novel triplex gene editing platform to potentially cure devastating genetic diseases, with initial focus on sickle cell disease and cystic fibrosis. The elegance of triplex gene editing lies in its ability to harness natural, high-fidelity DNA repair mechanisms, and its independence from the requirement for exogenous nucleases and viral vectors. <br><b>Specialties:</b> Not listed<br><b><a href="https://www.linkedin.com/company/trucode-gene-repair-inc/about/">LinkedIn Profile</a></b><br><b><a href="http://www.trucodegene.com">Website</a></b><br><br><b>Company Name:</b> addmedica<br><b>Overview:</b> addmedica is a fast growing company focused on developing and marketing medical products for:
• rare diseases
• unmet medical needs in developed and emerging countries
• serious conditions

addmedica is dedicated to providing high medical value to physicians, patients, and managed care organizations, by developing, registering and marketing a wide range of medical products in several rare and debilitating diseases. 

addmedica’s strategic areas of interest comprise:
• wound healing, tissue growth and cell regeneration
• organ transplant, tissue replacement 
• genetic diseases, drug or substitute therapy, cell or gene therapy

addmedica, complying with its pharmaceutical status, is proud to offer a range of products in the fields of 
• lung transplantation,
• wound healing and deep burn care. 
• sickle cell disease (European Orphan Medicinal Product)

addmedica is constantly looking for new opportunities and partnerships, to develop and commercialize products with high medical demand. 

As a result of its know-how, its commitment and its positioning, addmedica is ready to become a leading firm for the provision of medical products and innovative techniques for rare diseases with the best quality of services.<br><b>Specialties:</b> transplantation, wound-healing, and sickle-cell-disease<br><b><a href="https://www.linkedin.com/company/addmedica/about/">LinkedIn Profile</a></b><br><b><a href="http://www.addmedica.com">Website</a></b><br><br><b>Company Name:</b> Aruvant<br><b>Overview:</b> Aruvant Sciences is a clinical-stage gene therapy company focused on hematological conditions, with an emphasis on helping patients suffering from sickle cell disease and β-thalassemia. ARU-1801, the lead candidate in Aruvant's pipeline, is an investigational lentiviral gene therapy for sickle cell disease and transfusion-dependent β-thalassemia. ARU-1801 incorporates a patented gene payload for a modified gamma-globin delivered into autologous stem cells via a proprietary vector construct, with the aim of restoring normal red blood cell function through increased levels of fetal hemoglobin. The high potency of the modified gamma globin enables ARU-1801 engraftment with only Reduced Intensity Conditioning (RIC).<br><b>Specialties:</b> Not listed<br><b><a href="https://www.linkedin.com/company/aruvant/">LinkedIn Profile</a></b><br><b><a href="http://aruvant.com/">Website</a></b><br><br><b>Company Name:</b> Editas Medicine<br><b>Overview:</b> What if you could repair broken genes? That is the question we ask ourselves every day at Editas Medicine. We’re a leading genome editing company focused on translating the power and promise of our proprietary genome editing systems into medicines to help transform the lives of people with genetically-defined diseases. Our goal is to discover, develop, manufacture, and commercialize transformative medicines for a range of serious diseases, including eye diseases, blood diseases, and cancer. 

We are a vibrant company full of hope, possibilities, and a belief that, working together as One Editas, we can truly revolutionize the development of medicines. We are on an important journey to unlock the full potential of genome editing technology. A journey fueled by our distinct culture, expert team of Editas Medicine ‘Editors’, and the patients we aspire to help around the world. Connect with us to hear about the tremendous progress and scientific advancements we’ve already made and the next breakthrough on the horizon. If you are ingenious, passionate and resilient, come join the revolution. Repairing broken genes is only the beginning. <br><b>Specialties:</b> Genome Editing, In Vivo (in body) Editing, Engineered Cell Medicines, Ocular Medicines, Genetic Diseases, Blood Diseases, Sickle Cell Disease, Cancer, Rare Diseases, and Immunogenetics<br><b><a href="https://www.linkedin.com/company/editas-medicine/about/">LinkedIn Profile</a></b><br><b><a href="http://www.editasmedicine.com">Website</a></b><br><br><br><a name="patents"></a><h2>Patents</h2> <br><br></body></html>